<h1 align="center">
üß¨üìù Awesome Biomolecule-Language Cross Modeling
</h1>
<div align="center">

[![](https://img.shields.io/badge/paper-arxiv2403.01528-red?style=plastic&logo=GitBook)](https://arxiv.org/abs/2403.01528)
[![Awesome](https://awesome.re/badge.svg)](https://awesome.re) ![Stars](https://img.shields.io/github/stars/QizhiPei/Awesome-Biomolecule-Language-Cross-Modeling?color=yellow&labelColor=555555)  ![Forks](https://img.shields.io/github/forks/QizhiPei/Awesome-Biomolecule-Language-Cross-Modeling?color=blue&label=Fork&labelColor=555555)
</div>

The repository for [Leveraging Biomolecule and Natural Language through Multi-Modal Learning: A Survey](https://arxiv.org/abs/2403.01528), including related models, datasets/benchmarks, and other resource links.

üî• **We will keep this repository updated**.

üåü **If you have a paper or resource you'd like to add, feel free to submit a pull request, open an issue, or email the author at <qizhipei@ruc.edu.cn>.**

<p align="center">
  <img src="figs/model_evolution_v1.png" width="960">
</p>


## Table of Content
- [Models](#models)
  - [Biotext](#biotext)
  - [Text + Molecule](#text--molecule)
  - [Text + Protein](#text--protein)
  - [Text + BioMulti](#text-biomulti)
- [Datasets & Benchmarks](#datasets--benchmarks)
- [Related Resources](#related-resources)
  - [Related Surveys & Evaluations](#related-surveys--evaluations) 
  - [Related Repositories](#related-repositories)
- [Acknowledgements](#acknowledgements)
---

## Models
- `BioText` *`Bioinformatics 2019`* **[BioBERT: a pre-trained biomedical language representation model for biomedical text mining](https://academic.oup.com/bioinformatics/article/36/4/1234/5566506)** |   [![Model](https://img.shields.io/badge/Model-5291C8?style=flat&logo=themodelsresource&labelColor=555555)](https://huggingface.co/dmis-lab) [![Stars](https://img.shields.io/github/stars/dmis-lab/biobert?color=yellow&style=social)](https://github.com/dmis-lab/biobert)
- `BioText` *`EMNLP IJCNLP 2019`* **[SciBERT: A Pretrained Language Model for Scientific Text](https://aclanthology.org/D19-1371.pdf)** |   [![Model](https://img.shields.io/badge/Model-5291C8?style=flat&logo=themodelsresource&labelColor=555555)](https://github.com/allenai/scibert) [![Stars](https://img.shields.io/github/stars/allenai/scibert?color=yellow&style=social)](https://github.com/allenai/scibert)
- `BioText` *`BioNLP@ACL 2019`* **[(BlueBERT) Transfer Learning in Biomedical Natural Language Processing: An Evaluation of BERT and ELMo on Ten Benchmarking Datasets](https://aclanthology.org/W19-5006.pdf)** |   [![Model](https://img.shields.io/badge/Model-5291C8?style=flat&logo=themodelsresource&labelColor=555555)](https://huggingface.co/bionlp) [![Stars](https://img.shields.io/github/stars/ncbi-nlp/bluebert?color=yellow&style=social)](https://github.com/ncbi-nlp/bluebert)
- `BioText` *`EMNLP 2020`* **[Bio-Megatron: Larger Biomedical Domain Language Model](https://aclanthology.org/2020.emnlp-main.379.pdf)** |   [![Stars](https://img.shields.io/github/stars/NVIDIA/NeMo?color=yellow&style=social)](https://github.com/NVIDIA/NeMo)
- `BioText` *`BioNLP@CHIL 2020`* **[ClinicalBERT: Modeling Clinical Notes and Predicting Hospital Readmission](https://arxiv.org/pdf/1904.05342.pdf)** |   [![Model](https://img.shields.io/badge/Model-5291C8?style=flat&logo=themodelsresource&labelColor=555555)](https://github.com/lindvalllab/clinicalXLNet) [![Stars](https://img.shields.io/github/stars/kexinhuang12345/clinicalBERT?color=yellow&style=social)](https://github.com/kexinhuang12345/clinicalBERT)
- `BioText` *`BioNLP@ACL 2021`* **[BioM-Transformers: Building Large Biomedical Language Models with BERT, ALBERT and ELECTRA](https://aclanthology.org/2021.bionlp-1.24.pdf)** |   [![Model](https://img.shields.io/badge/Model-5291C8?style=flat&logo=themodelsresource&labelColor=555555)](https://huggingface.co/sultan) [![Stars](https://img.shields.io/github/stars/salrowili/BioM-Transformers?color=yellow&style=social)](https://github.com/salrowili/BioM-Transformers)
- `BioText` *`HEALTH 2021`* **[(PubMedBERT) Domain-Specific Language Model Pretraining for Biomedical Natural Language Processing](https://dl.acm.org/doi/10.1145/3458754)** |   [![Model](https://img.shields.io/badge/Model-5291C8?style=flat&logo=themodelsresource&labelColor=555555)](https://huggingface.co/microsoft/BiomedNLP-BiomedBERT-base-uncased-abstract-fulltext) 
- `BioText` *`Arxiv 2021`* **[SciFive: a text-to-text transformer model for biomedical literature](https://arxiv.org/pdf/2106.03598.pdf)** |   [![Model](https://img.shields.io/badge/Model-5291C8?style=flat&logo=themodelsresource&labelColor=555555)](https://huggingface.co/razent) [![Stars](https://img.shields.io/github/stars/justinphan3110/SciFive?color=yellow&style=social)](https://github.com/justinphan3110/SciFive)
- `BioText` *`NeurIPS 2022`* **[(DRAGON) Deep Bidirectional Language-Knowledge Graph Pretraining](https://arxiv.org/pdf/2210.09338.pdf)** |   [![Model](https://img.shields.io/badge/Model-5291C8?style=flat&logo=themodelsresource&labelColor=555555)](https://github.com/michiyasunaga/dragon) [![Stars](https://img.shields.io/github/stars/michiyasunaga/dragon?color=yellow&style=social)](https://github.com/michiyasunaga/dragon)
- `BioText` *`ACL 2022`* **[LinkBERT: Pretraining Language Models with Document Links](https://aclanthology.org/2022.acl-long.551.pdf)** |   [![Model](https://img.shields.io/badge/Model-5291C8?style=flat&logo=themodelsresource&labelColor=555555)](https://huggingface.co/michiyasunaga) [![Stars](https://img.shields.io/github/stars/michiyasunaga/LinkBERT?color=yellow&style=social)](https://github.com/michiyasunaga/LinkBERT)
- `BioText` *`BioNLP@ACL 2022`* **[BioBART: Pretraining and Evaluation of A Biomedical Generative Language Model](https://aclanthology.org/2022.bionlp-1.9.pdf)** |   [![Model](https://img.shields.io/badge/Model-5291C8?style=flat&logo=themodelsresource&labelColor=555555)](https://huggingface.co/GanjinZero) [![Stars](https://img.shields.io/github/stars/GanjinZero/BioBART?color=yellow&style=social)](https://github.com/GanjinZero/BioBART)
- `BioText` *`Bioinformatics 2022`* **[BioGPT: Generative Pre-trained Transformer for Biomedical Text Generation and Mining](https://doi.org/10.1093/bib/bbac409)** |   [![Model](https://img.shields.io/badge/Model-5291C8?style=flat&logo=themodelsresource&labelColor=555555)](https://huggingface.co/microsoft/biogpt) [![Stars](https://img.shields.io/github/stars/microsoft/BioGPT?color=yellow&style=social)](https://github.com/microsoft/BioGPT)
- `BioText` *`Arxiv 2022`* **[GatorTron: A Large Clinical Language Model to Unlock Patient Information from Unstructured Electronic Health Records](https://arxiv.org/pdf/2203.03540.pdf)** |   [![Stars](https://img.shields.io/github/stars/NVIDIA/Megatron-LM?color=yellow&style=social)](https://github.com/NVIDIA/Megatron-LM)
- `BioText` *`Nature 2023`* **[Large language models encode clinical knowledge](https://www.nature.com/articles/s41586-023-06291-2.pdf)** |   
- `BioText` *`ACL 2023`* **[(ScholarBERT) The Diminishing Returns of Masked Language Models to Science](https://aclanthology.org/2023.findings-acl.82.pdf)** |   [![Model](https://img.shields.io/badge/Model-5291C8?style=flat&logo=themodelsresource&labelColor=555555)](https://huggingface.co/globuslabs/ScholarBERT) 
- `BioText` *`Arxiv 2023`* **[PMC-LLaMA: Further Finetuning LLaMA on Medical Papers](https://arxiv.org/pdf/2304.14454.pdf)** |   [![Model](https://img.shields.io/badge/Model-5291C8?style=flat&logo=themodelsresource&labelColor=555555)](https://github.com/chaoyi-wu/PMC-LLaMA) [![Stars](https://img.shields.io/github/stars/chaoyi-wu/PMC-LLaMA?color=yellow&style=social)](https://github.com/chaoyi-wu/PMC-LLaMA)
- `BioText` *`Arxiv 2023`* **[BioMedGPT: Open Multimodal Generative Pre-trained Transformer for BioMedicine](https://arxiv.org/pdf/2308.09442v2.pdf)** |   [![Model](https://img.shields.io/badge/Model-5291C8?style=flat&logo=themodelsresource&labelColor=555555)](https://huggingface.co/PharMolix/BioMedGPT-LM-7B) [![Stars](https://img.shields.io/github/stars/PharMolix/OpenBioMed?color=yellow&style=social)](https://github.com/PharMolix/OpenBioMed)
- `BioText` *`Arxiv 2023`* **[(GatortronGPT) A study of generative large language model for medical research and healthcare](https://arxiv.org/pdf/2305.13523.pdf)** |   [![Stars](https://img.shields.io/github/stars/NVIDIA/Megatron-LM?color=yellow&style=social)](https://github.com/NVIDIA/Megatron-LM)
- `BioText` *`Arxiv 2023`* **[Clinical Camel: An Open-Source Expert-Level Medical Language Model with Dialogue-Based Knowledge Encoding](https://arxiv.org/pdf/2305.12031.pdf)** |   [![Model](https://img.shields.io/badge/Model-5291C8?style=flat&logo=themodelsresource&labelColor=555555)](https://huggingface.co/wanglab/ClinicalCamel-70B) [![Stars](https://img.shields.io/github/stars/bowang-lab/clinical-camel?color=yellow&style=social)](https://github.com/bowang-lab/clinical-camel)
- `BioText` *`Arxiv 2023`* **[MEDITRON-70B: Scaling Medical Pretraining for Large Language Models](https://arxiv.org/pdf/2311.16079.pdf)** |   [![Model](https://img.shields.io/badge/Model-5291C8?style=flat&logo=themodelsresource&labelColor=555555)](https://huggingface.co/epfl-llm/meditron-70b) [![Stars](https://img.shields.io/github/stars/epfLLM/meditron?color=yellow&style=social)](https://github.com/epfLLM/meditron)
- `BioText` *`Arxiv 2023`* **[BioinspiredLLM: Conversational Large Language Model for the Mechanics of Biological and Bio-inspired Materials](https://arxiv.org/pdf/2309.08788.pdf)** |   [![Model](https://img.shields.io/badge/Model-5291C8?style=flat&logo=themodelsresource&labelColor=555555)](https://huggingface.co/lamm-mit/BioinspiredLLM) 
- `BioText` *`Arxiv 2023`* **[ClinicalGPT: Large Language Models Finetuned with Diverse Medical Data and Comprehensive Evaluation](https://arxiv.org/pdf/2306.09968.pdf)** |   [![Model](https://img.shields.io/badge/Model-5291C8?style=flat&logo=themodelsresource&labelColor=555555)](https://huggingface.co/medicalai/ClinicalGPT-base-zh) 
- `BioText` *`Arxiv 2023`* **[MedAlpaca - An Open-Source Collection of Medical Conversational AI Models and Training Data](https://arxiv.org/pdf/2304.08247.pdf)** |   [![Model](https://img.shields.io/badge/Model-5291C8?style=flat&logo=themodelsresource&labelColor=555555)](https://huggingface.co/medalpaca) [![Stars](https://img.shields.io/github/stars/kbressem/medAlpaca?color=yellow&style=social)](https://github.com/kbressem/medAlpaca)
- `BioText` *`Arxiv 2024`* **[SciGLM: Training Scientific Language Models with Self-Reflective Instruction Annotation and Tuning](https://arxiv.org/pdf/2401.07950.pdf)** |   [![Model](https://img.shields.io/badge/Model-5291C8?style=flat&logo=themodelsresource&labelColor=555555)](https://huggingface.co/zd21/SciGLM-6B) [![Stars](https://img.shields.io/github/stars/THUDM/SciGLM?color=yellow&style=social)](https://github.com/THUDM/SciGLM)
- `BioText` *`Arxiv 2024`* **[BioMedLM: A 2.7B Parameter Language Model Trained On Biomedical Text](https://arxiv.org/pdf/2403.18421v1.pdf)** |   [![Model](https://img.shields.io/badge/Model-5291C8?style=flat&logo=themodelsresource&labelColor=555555)](https://huggingface.co/stanford-crfm/BioMedLM) [![Stars](https://img.shields.io/github/stars/stanford-crfm/BioMedLM?color=yellow&style=social)](https://github.com/stanford-crfm/BioMedLM)
- `BioText` *`ACL 2024`* **[BioMistral: A Collection of Open-Source Pretrained Large Language Models for Medical Domains](https://arxiv.org/pdf/2402.10373.pdf)** |   [![Model](https://img.shields.io/badge/Model-5291C8?style=flat&logo=themodelsresource&labelColor=555555)](https://huggingface.co/BioMistral) 
- `Text + Molecule` *`EMNLP 2021`* **[Text2Mol: Cross-Modal Molecule Retrieval with Natural Language Queries](https://aclanthology.org/2021.emnlp-main.47.pdf)** |   [![Stars](https://img.shields.io/github/stars/cnedwards/text2mol?color=yellow&style=social)](https://github.com/cnedwards/text2mol)
- `Text + Molecule` *`EMNLP 2022`* **[(MolT5) Translation between Molecules and Natural Language](https://aclanthology.org/2022.emnlp-main.26.pdf)** |   [![Model](https://img.shields.io/badge/Model-5291C8?style=flat&logo=themodelsresource&labelColor=555555)](https://huggingface.co/laituan245) [![Stars](https://img.shields.io/github/stars/blender-nlp/MolT5?color=yellow&style=social)](https://github.com/blender-nlp/MolT5)
- `Text + Molecule` *`Nature Communications 2022`* **[(KV-PLM) A deep-learning system bridging molecule structure and biomedical text with comprehension comparable to human professionals](https://www.nature.com/articles/s41467-022-28494-3.pdf)** |   [![Model](https://img.shields.io/badge/Model-5291C8?style=flat&logo=themodelsresource&labelColor=555555)](https://drive.google.com/drive/folders/1xig3-3JG63kR-Xqj1b9wkPEdxtfD_4IX) [![Stars](https://img.shields.io/github/stars/thunlp/KV-PLM?color=yellow&style=social)](https://github.com/thunlp/KV-PLM)
- `Text + Molecule` *`Arxiv 2022`* **[(MoMu) A Molecular Multimodal Foundation Model Associating Molecule Graphs with Natural Language](https://arxiv.org/pdf/2209.05481.pdf)** |   [![Model](https://img.shields.io/badge/Model-5291C8?style=flat&logo=themodelsresource&labelColor=555555)](https://github.com/BingSu12/MoMu) [![Stars](https://img.shields.io/github/stars/BingSu12/MoMu?color=yellow&style=social)](https://github.com/BingSu12/MoMu)
- `Text + Molecule` *`ICML 2023`* **[(Text+Chem T5) Unifying Molecular and Textual Representations via Multi-task Language Modelling](https://proceedings.mlr.press/v202/christofidellis23a/christofidellis23a.pdf)** |   [![Model](https://img.shields.io/badge/Model-5291C8?style=flat&logo=themodelsresource&labelColor=555555)](https://huggingface.co/GT4SD) [![Stars](https://img.shields.io/github/stars/GT4SD/multitask_text_and_chemistry_t5?color=yellow&style=social)](https://github.com/GT4SD/multitask_text_and_chemistry_t5)
- `Text + Molecule` *`ICML 2023`* **[(CLAMP) Enhancing Activity Prediction Models in Drug Discovery with the Ability to Understand Human Language](https://proceedings.mlr.press/v202/seidl23a/seidl23a.pdf)** |   [![Model](https://img.shields.io/badge/Model-5291C8?style=flat&logo=themodelsresource&labelColor=555555)](https://github.com/ml-jku/clamp) [![Stars](https://img.shields.io/github/stars/ml-jku/clamp?color=yellow&style=social)](https://github.com/ml-jku/clamp)
- `Text + Molecule` *`NeurIPS 2023`* **[GIMLET: A Unified Graph-Text Model for Instruction-Based Molecule Zero-Shot Learning](https://openreview.net/pdf?id=Tt6DrRCgJV)** |   [![Model](https://img.shields.io/badge/Model-5291C8?style=flat&logo=themodelsresource&labelColor=555555)](https://huggingface.co/haitengzhao/gimlet) [![Stars](https://img.shields.io/github/stars/zhao-ht/GIMLET?color=yellow&style=social)](https://github.com/zhao-ht/GIMLET)
- `Text + Molecule` *`AI4D3@NeurIPS 2023`* **[MoleculeGPT: Instruction Following Large Language Models for Molecular Property Prediction](https://ai4d3.github.io/papers/34.pdf)** |   
- `Text + Molecule` *`Datasets&Benchmarks@NeurIPS 2023`* **[(ChemLLMBench) What indeed can GPT models do in chemistry? A comprehensive benchmark on eight tasks](https://openreview.net/pdf?id=1ngbR3SZHW)** |   [![Stars](https://img.shields.io/github/stars/ChemFoundationModels/ChemLLMBench?color=yellow&style=social)](https://github.com/ChemFoundationModels/ChemLLMBench)
- `Text + Molecule` *`ACL 2023`* **[MolXPT: Wrapping Molecules with Text for Generative Pre-training](https://aclanthology.org/2023.acl-short.138.pdf)** |   
- `Text + Molecule` *`EMNLP 2023`* **[(TextReact) Predictive Chemistry Augmented with Text Retrieval](https://aclanthology.org/2023.emnlp-main.784.pdf)** |   [![Stars](https://img.shields.io/github/stars/thomas0809/textreact?color=yellow&style=social)](https://github.com/thomas0809/textreact)
- `Text + Molecule` *`EMNLP 2023`* **[MolCA: Molecular Graph-Language Modeling with Cross-Modal Projector and Uni-Modal Adapter](https://aclanthology.org/2023.emnlp-main.966v2.pdf)** |   [![Model](https://img.shields.io/badge/Model-5291C8?style=flat&logo=themodelsresource&labelColor=555555)](https://ufile.io/6vffm5bg) [![Stars](https://img.shields.io/github/stars/eltociear/MolCA?color=yellow&style=social)](https://github.com/eltociear/MolCA)
- `Text + Molecule` *`EMNLP 2023`* **[ReLM: Leveraging Language Models for Enhanced Chemical Reaction Prediction](https://aclanthology.org/2023.findings-emnlp.366.pdf)** |   [![Stars](https://img.shields.io/github/stars/syr-cn/ReLM?color=yellow&style=social)](https://github.com/syr-cn/ReLM)
- `Text + Molecule` *`Nature Machine Intelligence 2023`* **[(MoleculeSTM) Multi-modal Molecule Structure-text Model for Text-based Retrieval and Editing](https://www.nature.com/articles/s42256-023-00759-6.pdf)** |   [![Model](https://img.shields.io/badge/Model-5291C8?style=flat&logo=themodelsresource&labelColor=555555)](https://huggingface.co/chao1224/MoleculeSTM) [![Stars](https://img.shields.io/github/stars/chao1224/MoleculeSTM?color=yellow&style=social)](https://github.com/chao1224/MoleculeSTM)
- `Text + Molecule` *`IEEE TAI 2023`* **[(AMAN) Adversarial Modality Alignment Network for Cross-Modal Molecule Retrieval](https://ieeexplore.ieee.org/document/10063974)** |   [![Stars](https://img.shields.io/github/stars/NicoleBonnie/AMAN?color=yellow&style=social)](https://github.com/NicoleBonnie/AMAN)
- `Text + Molecule` *`BioRxiv 2023`* **[MolLM: A Unified Language Model for Integrating Biomedical Text with 2D and 3D Molecular Representations](https://www.biorxiv.org/content/10.1101/2023.11.25.568656v2.full.pdf)** |   [![Model](https://img.shields.io/badge/Model-5291C8?style=flat&logo=themodelsresource&labelColor=555555)](https://drive.google.com/drive/folders/17XhqdsDOxiT8PEDLHdsLPKf62PXPmbms) [![Stars](https://img.shields.io/github/stars/gersteinlab/MolLM?color=yellow&style=social)](https://github.com/gersteinlab/MolLM)
- `Text + Molecule` *`Arxiv 2023`* **[(MolReGPT) Empowering Molecule Discovery for Molecule-Caption Translation with Large Language Models: A ChatGPT Perspective](https://arxiv.org/pdf/2306.06615.pdf)** |   [![Stars](https://img.shields.io/github/stars/phenixace/MolReGPT?color=yellow&style=social)](https://github.com/phenixace/MolReGPT)
- `Text + Molecule` *`Arxiv 2023`* **[(CaR) Can Large Language Models Empower Molecular Property Prediction?](https://arxiv.org/pdf/2307.07443.pdf)** |   [![Stars](https://img.shields.io/github/stars/ChnQ/LLM4Mol?color=yellow&style=social)](https://github.com/ChnQ/LLM4Mol)
- `Text + Molecule` *`Arxiv 2023`* **[MolFM: A Multimodal Molecular Foundation Model](https://arxiv.org/pdf/2307.09484.pdf)** |   [![Model](https://img.shields.io/badge/Model-5291C8?style=flat&logo=themodelsresource&labelColor=555555)](https://pan.baidu.com/share/init?surl=iAMBkuoZnNAylhopP5OgEg\&pwd=7a6b) [![Stars](https://img.shields.io/github/stars/PharMolix/OpenBioMed?color=yellow&style=social)](https://github.com/PharMolix/OpenBioMed)
- `Text + Molecule` *`Arxiv 2023`* **[(ChatMol) Interactive Molecular Discovery with Natural Language](https://arxiv.org/pdf/2306.11976.pdf)** |   [![Model](https://img.shields.io/badge/Model-5291C8?style=flat&logo=themodelsresource&labelColor=555555)](https://drive.google.com/drive/folders/1I-LcBE0emj8p1W6WFCbEYajTADGN1RuC) [![Stars](https://img.shields.io/github/stars/Ellenzzn/ChatMol?color=yellow&style=social)](https://github.com/Ellenzzn/ChatMol)
- `Text + Molecule` *`Arxiv 2023`* **[InstructMol: Multi-Modal Integration for Building a Versatile and Reliable Molecular Assistant in Drug Discovery](https://arxiv.org/pdf/2311.16208.pdf)** |   [![Stars](https://img.shields.io/github/stars/IDEA-XL/InstructMol?color=yellow&style=social)](https://github.com/IDEA-XL/InstructMol)
- `Text + Molecule` *`Arxiv 2023`* **[ChemCrow: Augmenting large-language models with chemistry tools](https://arxiv.org/pdf/2304.05376.pdf)** |   [![Stars](https://img.shields.io/github/stars/ur-whitelab/chemcrow-public?color=yellow&style=social)](https://github.com/ur-whitelab/chemcrow-public)
- `Text + Molecule` *`Arxiv 2023`* **[GPT-MolBERTa: GPT Molecular Features Language Model for molecular property prediction](https://arxiv.org/pdf/2310.03030.pdf)** |   [![Stars](https://img.shields.io/github/stars/Suryanarayanan-Balaji/GPT-MolBERTa?color=yellow&style=social)](https://github.com/Suryanarayanan-Balaji/GPT-MolBERTa)
- `Text + Molecule` *`Arxiv 2023`* **[nach0: Multimodal Natural and Chemical Languages Foundation Model](https://arxiv.org/pdf/2311.12410.pdf)** |   
- `Text + Molecule` *`Arxiv 2023`* **[DrugChat: Towards Enabling ChatGPT-Like Capabilities on Drug Molecule Graphs](https://arxiv.org/pdf/2309.03907.pdf)** |   [![Stars](https://img.shields.io/github/stars/UCSD-AI4H/drugchat?color=yellow&style=social)](https://github.com/UCSD-AI4H/drugchat)
- `Text + Molecule` *`AAAI 2024`* **[(Ada/Aug-T5) From Artificially Real to Real: Leveraging Pseudo Data from Large Language Models for Low-Resource Molecule Discovery](https://arxiv.org/pdf/2309.05203.pdf)** |   [![Model](https://img.shields.io/badge/Model-5291C8?style=flat&logo=themodelsresource&labelColor=555555)](https://huggingface.co/SCIR-HI) [![Stars](https://img.shields.io/github/stars/SCIR-HI/ArtificiallyR2R?color=yellow&style=social)](https://github.com/SCIR-HI/ArtificiallyR2R)
- `Text + Molecule` *`AAAI 2024`* **[MolTailor: Tailoring Chemical Molecular Representation to Specific Tasks via Text Prompts](https://arxiv.org/pdf/2401.11403.pdf)** |   [![Stars](https://img.shields.io/github/stars/SCIR-HI/MolTailor?color=yellow&style=social)](https://github.com/SCIR-HI/MolTailor)
- `Text + Molecule` *`AAAI 2024`* **[(TGM-DLM) Text-Guided Molecule Generation with Diffusion Language Model](https://arxiv.org/pdf/2402.13040.pdf)** |   [![Stars](https://img.shields.io/github/stars/Deno-V/tgm-dlm?color=yellow&style=social)](https://github.com/Deno-V/tgm-dlm)
- `Text + Molecule` *`Computers in Biology and Medicine 2024`* **[GIT-Mol: A Multi-modal Large Language Model for Molecular Science with Graph, Image, and Text](https://doi.org/10.1016/j.compbiomed.2024.108073)** |   [![Stars](https://img.shields.io/github/stars/ai-hpc-research-team/git-mol?color=yellow&style=social)](https://github.com/ai-hpc-research-team/git-mol)
- `Text + Molecule` *`Chemical Science 2024`* **[PolyNC: a natural and chemical language model for the prediction of unified polymer properties](https://pubs.rsc.org/en/content/articlepdf/2024/sc/d3sc05079c)** |   [![Model](https://img.shields.io/badge/Model-5291C8?style=flat&logo=themodelsresource&labelColor=555555)](https://huggingface.co/hkqiu/PolyNC) [![Stars](https://img.shields.io/github/stars/HKQiu/Unified_ML4Polymers?color=yellow&style=social)](https://github.com/HKQiu/Unified_ML4Polymers)
- `Text + Molecule` *`Arxiv 2024`* **[MolTC: Towards Molecular Relational Modeling In Language Models](https://arxiv.org/pdf/2402.03781.pdf)** |   [![Stars](https://img.shields.io/github/stars/MangoKiller/MolTC?color=yellow&style=social)](https://github.com/MangoKiller/MolTC)
- `Text + Molecule` *`Arxiv 2024`* **[T-Rex: Text-assisted Retrosynthesis Prediction](https://arxiv.org/pdf/2401.14637.pdf)** |   [![Stars](https://img.shields.io/github/stars/lauyikfung/T-Rex?color=yellow&style=social)](https://github.com/lauyikfung/T-Rex)
- `Text + Molecule` *`COLM 2024`* **[LlaSMol: Advancing Large Language Models for Chemistry with a Large-Scale, Comprehensive, High-Quality Instruction Tuning Dataset](https://arxiv.org/pdf/2402.09391.pdf)** |   [![Model](https://img.shields.io/badge/Model-5291C8?style=flat&logo=themodelsresource&labelColor=555555)](https://huggingface.co/osunlp/LLM4Chem) [![Stars](https://img.shields.io/github/stars/OSU-NLP-Group/LLM4Chem?color=yellow&style=social)](https://github.com/OSU-NLP-Group/LLM4Chem)
- `Text + Molecule` *`Arxiv 2024`* **[(Drug-to-indication) Emerging Opportunities of Using Large Language Models for Translation Between Drug Molecules and Indications](https://arxiv.org/pdf/2402.09588.pdf)** |   [![Stars](https://img.shields.io/github/stars/PittNAIL/drug-to-indication?color=yellow&style=social)](https://github.com/PittNAIL/drug-to-indication)
- `Text + Molecule` *`Arxiv 2024`* **[ChemDFM: Dialogue Foundation Model for Chemistry](https://arxiv.org/pdf/2401.14818.pdf)** |   
- `Text + Molecule` *`Briefings in Bioinformatics 2024`* **[DrugAssist: A Large Language Model for Molecule Optimization](https://academic.oup.com/bib/article/26/1/bbae693/7942355)** |   [![Stars](https://img.shields.io/github/stars/blazerye/DrugAssist?color=yellow&style=social)](https://github.com/blazerye/DrugAssist)
- `Text + Molecule` *`Arxiv 2024`* **[ChemLLM: A Chemical Large Language Model](https://arxiv.org/pdf/2402.06852.pdf)** |   [![Model](https://img.shields.io/badge/Model-5291C8?style=flat&logo=themodelsresource&labelColor=555555)](https://huggingface.co/AI4Chem/ChemLLM-7B-Chat) 
- `Text + Molecule` *`OpenReview`* **[(TEDMol) Text-guided Diffusion Model for 3D Molecule Generation](https://openreview.net/pdf?id=FdUloEgBSE)** |   
- `Text + Molecule` *`Bioinformatics 2025`* **[(3DToMolo) Sculpting Molecules in 3D: A Flexible Substructure Aware Framework for Text-Oriented Molecular Optimization](https://arxiv.org/pdf/2403.03425.pdf)** |   
- `Text + Molecule` *`IEEE TKDE‚Äå 2024`* **[(ICMA) Large Language Models are In-Context Molecule Learners](https://arxiv.org/pdf/2403.04197.pdf)** |   [![Model](https://img.shields.io/badge/Model-5291C8?style=flat&logo=themodelsresource&labelColor=555555)](https://huggingface.co/phenixace) 
- `Text + Molecule` *`Arxiv 2024`* **[(LLMaMol) Benchmarking Large Language Models for Molecule Prediction Tasks](https://arxiv.org/pdf/2403.05075.pdf)** |   [![Stars](https://img.shields.io/github/stars/zhiqiangzhongddu/LLMaMol?color=yellow&style=social)](https://github.com/zhiqiangzhongddu/LLMaMol)
- `Text + Molecule` *`Arxiv 2024`* **[3M-Diffusion: Latent Multi-Modal Diffusion for Text-Guided Generation of Molecular Graphs](https://arxiv.org/pdf/2403.07179.pdf)** |   [![Stars](https://img.shields.io/github/stars/huaishengzhu/3MDiffusion?color=yellow&style=social)](https://github.com/huaishengzhu/3MDiffusion)
- `Text + Molecule` *`Biology 2024`* **[(TSMMG) Instruction Multi-Constraint Molecular Generation Using a Teacher-Student Large Language Model](https://arxiv.org/pdf/2403.13244.pdf)** |   [![Stars](https://img.shields.io/github/stars/HHW-zhou/TSMMG?color=yellow&style=social)](https://github.com/HHW-zhou/TSMMG)
- `Text + Molecule` *`Eng Appl Artif Intell`* **[(SLM4CRP) A Self-feedback Knowledge Elicitation Approach for Chemical Reaction Predictions](https://arxiv.org/pdf/2404.09606.pdf)** |   [![Stars](https://img.shields.io/github/stars/AI-HPC-Research-Team/SLM4CRP?color=yellow&style=social)](https://github.com/AI-HPC-Research-Team/SLM4CRP)
- `Text + Molecule` *`ICLR 2025`* **[Atomas: Hierarchical Alignment on Molecule-Text for Unified Molecule Understanding and Generation](https://arxiv.org/pdf/2404.16880.pdf)** |   [![Stars](https://img.shields.io/github/stars/yikunpku/Atomas?color=yellow&style=social)](https://github.com/yikunpku/Atomas)
- `Text + Molecule` *`ACL 2024`* **[ReactXT: Understanding Molecular"Reaction-ship"via Reaction-Contextualized Molecule-Text Pretraining](https://arxiv.org/pdf/2405.14225)** |   [![Stars](https://img.shields.io/github/stars/syr-cn/ReactXT?color=yellow&style=social)](https://github.com/syr-cn/ReactXT)
- `Text + Molecule` *`L+M 2024`* **[ALMol: Aligned Language-Molecule Translation LLMs through Offline Preference Contrastive Optimisation](https://aclanthology.org/2024.langmol-1.3/)** |   [![Stars](https://img.shields.io/github/stars/REAL-Lab-NU/Awesome-LLM-Centric-Molecular-Discovery?color=yellow&style=social)](https://github.com/REAL-Lab-NUgithub.com/REAL-Lab-NU/Awesome-LLM-Centric-Molecular-Discovery)
- `Text + Molecule` *`Arxiv 2024`* **[LDMol: Text-Conditioned Molecule Diffusion Model Leveraging Chemically Informative Latent Space](https://arxiv.org/pdf/2405.17829)** |   [![Stars](https://img.shields.io/github/stars/jinhojsk515/LDMol?color=yellow&style=social)](https://github.com/jinhojsk515/LDMol)
- `Text + Molecule` *`Arxiv 2024`* **[DrugLLM: Open Large Language Model for Few-shot Molecule Generation](https://arxiv.org/pdf/2405.06690)** |   
- `Text + Molecule` *`Arxiv 2024`* **[(HI-Mol) Data-Efficient Molecular Generation with Hierarchical Textual Inversion](https://arxiv.org/abs/2405.02845)** |   [![Stars](https://img.shields.io/github/stars/Seojin-Kim/HI-Mol?color=yellow&style=social)](https://github.com/Seojin-Kim/HI-Mol)
- `Text + Molecule` *`KDD 2024`* **[(MV-Mol) Learning Multi-view Molecular Representations with Structured and Unstructured Knowledge](https://arxiv.org/pdf/2406.09841)** |   [![Model](https://img.shields.io/badge/Model-5291C8?style=flat&logo=themodelsresource&labelColor=555555)](https://drive.google.com/drive/folders/1cd1EZTuyNOCLRt2Vr0JgXh3kWm9f2qwU) [![Stars](https://img.shields.io/github/stars/icycookies/MV-Mol?color=yellow&style=social)](https://github.com/icycookies/MV-Mol)
- `Text + Molecule` *`NLPCC 2024`* **[DRAK: Unlocking Molecular Insights with Domain-Specific Retrieval-Augmented Knowledge in LLMs](https://arxiv.org/pdf/2406.18535)** |   
- `Text + Molecule` *`Arxiv 2024`* **[HIGHT: Hierarchical Graph Tokenization for Graph-Language Alignment](https://arxiv.org/pdf/2406.14021)** |   [![Stars](https://img.shields.io/github/stars/LFhase/HIGHT?color=yellow&style=social)](https://github.com/LFhase/HIGHT)
- `Text + Molecule` *`EMNLP 2024`* **[PRESTO: Progressive Pretraining Enhances Synthetic Chemistry Outcomes](https://arxiv.org/pdf/2406.13193v1)** |   [![Stars](https://img.shields.io/github/stars/IDEA-XL/PRESTO?color=yellow&style=social)](https://github.com/IDEA-XL/PRESTO)
- `Text + Molecule` *`Arxiv 2024`* **[3D-MolT5: Towards Unified 3D Molecule-Text Modeling with 3D Molecular Tokenization](https://arxiv.org/pdf/2406.05797)** |   [![Stars](https://img.shields.io/github/stars/QizhiPei/3D-MolT5?color=yellow&style=social)](https://github.com/QizhiPei/3D-MolT5)
- `Text + Molecule` *`Arxiv 2024`* **[MolecularGPT: Open Large Language Model (LLM) for Few-Shot Molecular Property Prediction](https://arxiv.org/pdf/2406.12950)** |   [![Model](https://img.shields.io/badge/Model-5291C8?style=flat&logo=themodelsresource&labelColor=555555)](https://huggingface.co/YuyanLiu/MolecularGPT) [![Stars](https://img.shields.io/github/stars/NYUSHCS/MolecularGPT?color=yellow&style=social)](https://github.com/NYUSHCS/MolecularGPT)
- `Text + Molecule` *`Arxiv 2024`* **[MolX: Enhancing Large Language Models for Molecular Learning with A Multi-Modal Extension](https://arxiv.org/pdf/2406.06777)** |   
- `Text + Molecule` *`Arxiv 2024`* **[(AMOLE) Vision Language Model is NOT All You Need: Augmentation Strategies for Molecule Language Models](https://arxiv.org/pdf/2407.09043)** |   [![Stars](https://img.shields.io/github/stars/Namkyeong/AMOLE?color=yellow&style=social)](https://github.com/Namkyeong/AMOLE)
- `Text + Molecule` *`Arxiv 2024`* **[(Chemma-RC) Text-Augmented Multimodal LLMs for Chemical Reaction Condition Recommendation](https://arxiv.org/abs/2407.15141)** |   
- `Text + Molecule` *`ICLR 2024`* **[(SMILES-probing) Chemical Language Models Have Problems with Chemistry: A Case Study on Molecule Captioning Task](https://openreview.net/pdf?id=JoO6mtCLHD)** |   [![Stars](https://img.shields.io/github/stars/ChemistryLLMs/SMILES-probing?color=yellow&style=social)](https://github.com/ChemistryLLMs/SMILES-probing)
- `Text + Molecule` *`Arxiv 2024`* **[UniMoT: Unified Molecule-Text Language Model with Discrete Token Representation](https://arxiv.org/pdf/2408.00863)** |   [![Stars](https://img.shields.io/github/stars/Uni-MoT/uni-mot.github.io?color=yellow&style=social)](https://github.com/Uni-MoT/uni-mot.github.io)
- `Text + Molecule` *`ECAI 2025`* **[(UTGDiff) Instruction-Based Molecular Graph Generation with Unified Text-Graph Diffusion Model](https://arxiv.org/abs/2408.09896)** |   [![Model](https://img.shields.io/badge/Model-5291C8?style=flat&logo=themodelsresource&labelColor=555555)](https://drive.google.com/drive/folders/18EqQ7MDHesmtiMiZz2o09PyeSwyf0hXb) [![Stars](https://img.shields.io/github/stars/ran1812/UTGDiff?color=yellow&style=social)](https://github.com/ran1812/UTGDiff)
- `Text + Molecule` *`ACL 2024`* **[Mol2Lang-VLM: Vision- and Text-Guided Generative Pre-trained Language Models for Advancing Molecule Captioning through Multimodal Fusion](https://aclanthology.org/2024.langmol-1.12/)** |   [![Stars](https://img.shields.io/github/stars/nhattruongpham/mol-lang-bridge?color=yellow&style=social)](https://github.com/nhattruongpham/mol-lang-bridge/tree/mol2lang/)
- `Text + Molecule` *`ACL 2024`* **[Lang2Mol-Diff: A Diffusion-Based Generative Model for Language-to-Molecule Translation Leveraging SELFIES Representation](https://aclanthology.org/2024.langmol-1.15/)** |   [![Stars](https://img.shields.io/github/stars/nhattruongpham/mol-lang-bridge?color=yellow&style=social)](https://github.com/nhattruongpham/mol-lang-bridge/tree/lang2mol/)
- `Text + Molecule` *`ACL 2024`* **[(Self-Augmentation) Enhancing Cross Text-Molecule Learning by Self-Augmentation](https://aclanthology.org/2024.findings-acl.569/)** |   
- `Text + Molecule` *`ACL 2024`* **[MTSwitch: A Web-based System for Translation between Molecules and Texts](https://aclanthology.org/2024.inlg-demos.2.pdf)** |   [![Stars](https://img.shields.io/github/stars/hanninaa/MTSwitch?color=yellow&style=social)](https://github.com/hanninaa/MTSwitch)
- `Text + Molecule` *`NeurIPS 2024`* **[SmileyLlama: Modifying Large Language Models for Directed Chemical Space Exploration](https://arxiv.org/abs/2409.02231)** |   
- `Text + Molecule` *`NeurIPS 2024`* **[(MoleculeSTM) Geometry-text Multi-modal Foundation Model for Reactivity-oriented Molecule Editing](https://openreview.net/forum?id=A9FlQMKxJ4)** |   [![Model](https://img.shields.io/badge/Model-5291C8?style=flat&logo=themodelsresource&labelColor=555555)](https://huggingface.co/chao1224/MoleculeSTM/tree/main) [![Stars](https://img.shields.io/github/stars/chao1224/MoleculeSTM?color=yellow&style=social)](https://github.com/chao1224/MoleculeSTM)
- `Text + Molecule` *`ACL 2025`* **[(MSR) Structural Reasoning Improves Molecular Understanding of LLM](https://arxiv.org/abs/2410.05610)** |   [![Stars](https://img.shields.io/github/stars/yunhuijang/MSR?color=yellow&style=social)](https://github.com/yunhuijang/MSR)
- `Text + Molecule` *`Arxiv 2024`* **[(TransDLM) Text-Guided Multi-Property Molecular Optimization with a Diffusion Language Model](https://arxiv.org/abs/2410.13597)** |   [![Stars](https://img.shields.io/github/stars/Cello2195/TransDLM?color=yellow&style=social)](https://github.com/Cello2195/TransDLM)
- `Text + Molecule` *`Arxiv 2024`* **[Can LLMs Generate Diverse Molecules? Towards Alignment with Structural Diversity](https://arxiv.org/abs/2410.03138)** |   
- `Text + Molecule` *`JCIM 2025`* **[(ChemLML) Chemical Language Model Linker: blending text and molecules with modular adapters](https://arxiv.org/pdf/2410.20182)** |   [![Model](https://img.shields.io/badge/Model-5291C8?style=flat&logo=themodelsresource&labelColor=555555)](https://doi.org/10.5281/zenodo.11661517) [![Stars](https://img.shields.io/github/stars/gitter-lab/ChemLML?color=yellow&style=social)](https://github.com/gitter-lab/ChemLML)
- `Text + Molecule` *`NeurIPS 2024`* **[(Chemlactica) Small Molecule Optimization with Large Language Models](https://openreview.net/forum?id=nJCYKdRZXb)** |   [![Stars](https://img.shields.io/github/stars/yerevann/chemlactica?color=yellow&style=social)](https://github.com/yerevann/chemlactica)
- `Text + Molecule` *`NeurIPS 2024`* **[Question Rephrasing for Quantifying Uncertainty in Large Language Models: Applications in Molecular Chemistry Tasks](https://neurips.cc/media/neurips-2024/Slides/105558.pdf)** |   
- `Text + Molecule` *`NeurIPS 2024`* **[(LLaMo) Large Language Model-based Molecular Graph Assistant](https://proceedings.neurips.cc/paper_files/paper/2024/file/ee46288ab2aaf5c6e53aebebe719712c-Paper-Conference.pdf)** |   [![Stars](https://img.shields.io/github/stars/mlvlab/LLaMo?color=yellow&style=social)](https://github.com/mlvlab/LLaMo)
- `Text + Molecule` *`NeurIPS 2024`* **[(MolPuzzle) Can LLMs Solve Molecule Puzzles? A Multimodal Benchmark for Molecular Structure Elucidation](https://proceedings.neurips.cc/paper_files/paper/2024/hash/f2b9e8e7a36d43ddfd3d55113d56b1e0-Abstract-Datasets_and_Benchmarks_Track.html)** |   [![Stars](https://img.shields.io/github/stars/KehanGuo2/MolPuzzle?color=yellow&style=social)](https://github.com/KehanGuo2/MolPuzzle)
- `Text + Molecule` *`Arxiv 2024`* **[(M¬≥LLM) Exploring Hierarchical Molecular Graph Representation in Multimodal LLMs](https://arxiv.org/abs/2411.04708)** |   
- `Text + Molecule` *`Arxiv 2024`* **[MolReFlect: Towards In-Context Fine-grained Alignments between Molecules and Texts](https://arxiv.org/abs/2411.14721)** |   
- `Text + Molecule` *`ENMLP 2024`* **[(AMORE) Lost in Translation: Chemical Language Models and the Misunderstanding of Molecule Structures](https://aclanthology.org/2024.findings-emnlp.760/)** |   [![Stars](https://img.shields.io/github/stars/ChemistryLLMs/AMORE?color=yellow&style=social)](https://github.com/ChemistryLLMs/AMORE)
- `Text + Molecule` *`BIBM 2024`* **[GeomCLIP: Contrastive Geometry-Text Pre-training for Molecules](https://ieeexplore.ieee.org/abstract/document/10822346)** |   [![Stars](https://img.shields.io/github/stars/xiaocui3737/GeomCLIP?color=yellow&style=social)](https://github.com/xiaocui3737/GeomCLIP)
- `Text + Molecule` *`BIBM 2024`* **[(CMTMR) Towards Cross-Modal Text-Molecule Retrieval with Better Modality Alignment](https://ieeexplore.ieee.org/abstract/document/10821722)** |   [![Stars](https://img.shields.io/github/stars/XMUDeepLIT/CMTMR?color=yellow&style=social)](https://github.com/XMUDeepLIT/CMTMR)
- `Text + Molecule` *`BIBM 2024`* **[(ORMA) Exploring Optimal Transport-Based Multi-Grained Alignments for Text-Molecule Retrieval](https://ieeexplore.ieee.org/abstract/document/10822800)** |   [![Stars](https://img.shields.io/github/stars/XMUDeepLIT/Orma?color=yellow&style=social)](https://github.com/XMUDeepLIT/Orma)
- `Text + Molecule` *`Arxiv 2024`* **[PEIT: Property Enhanced Instruction Tuning for Multi-task Molecular Generation with LLMs](https://arxiv.org/abs/2412.18084)** |   [![Model](https://img.shields.io/badge/Model-5291C8?style=flat&logo=themodelsresource&labelColor=555555)](https://huggingface.co/ccsalong/PEIT-LLM-LLaMa3.1-8B/tree/main) [![Stars](https://img.shields.io/github/stars/chenlong164/PEIT?color=yellow&style=social)](https://github.com/chenlong164/PEIT)
- `Text + Molecule` *`Arxiv 2024`* **[(HME) Navigating Chemical-Linguistic Sharing Space with Heterogeneous Molecular Encoding](https://arxiv.org/abs/2412.20888)** |   [![Stars](https://img.shields.io/github/stars/Lyu6PosHao/HME?color=yellow&style=social)](https://github.com/Lyu6PosHao/HME)
- `Text + Molecule` *`ICLR 2025`* **[(Llamole) Multimodal Large Language3D-MOLT5 Models for Inverse Molecular Design with Retrosynthetic Planning](https://openreview.net/forum?id=rQ7fz9NO7f)** |   [![Model](https://img.shields.io/badge/Model-5291C8?style=flat&logo=themodelsresource&labelColor=555555)](https://huggingface.co/collections/liuganghuggingface/llamole-collection) [![Stars](https://img.shields.io/github/stars/liugangcode/Llamole?color=yellow&style=social)](https://github.com/liugangcode/Llamole)
- `Text + Molecule` *`ICLR 2025`* **[RetroInText: A Multimodal Large Language Model Enhanced Framework for Retrosynthetic Planning via In-Context Representation Learning](https://openreview.net/forum?id=J6e4hurEKd)** |   [![Stars](https://img.shields.io/github/stars/Kin-CL/RetroInText?color=yellow&style=social)](https://github.com/Kin-CL/RetroInText)
- `Text + Molecule` *`Arxiv 2025`* **[OCSU: Optical Chemical Structure Understanding for Molecule-centric Scientific Discovery](https://arxiv.org/abs/2501.15415)** |   [![Model](https://img.shields.io/badge/Model-5291C8?style=flat&logo=themodelsresource&labelColor=555555)](https://huggingface.co/PharMolix/Mol-VL-7B) [![Stars](https://img.shields.io/github/stars/PharMolix/OCSU?color=yellow&style=social)](https://github.com/PharMolix/OCSU)
- `Text + Molecule` *`NeurIPS 2025`* **[Omni-Mol: Multitask Molecular Model for Any-to-any Modalities](https://arxiv.org/abs/2502.01074)** |   [![Model](https://img.shields.io/badge/Model-5291C8?style=flat&logo=themodelsresource&labelColor=555555)](https://huggingface.co/datasets/CodeMagic/Omni-Mol-Dataset) [![Stars](https://img.shields.io/github/stars/1789336421/Omni-Mol?color=yellow&style=social)](https://github.com/1789336421/Omni-Mol)
- `Text + Molecule` *`Arxiv 2025`* **[Mol-LLM: Multimodal Generalist Molecular LLM with Improved Graph Utilization](https://arxiv.org/abs/2502.02810)** |   
- `Text + Molecule` *`NMI 2025`* **[(SLM4Mol) A Quantitative Analysis of Knowledge-Learning Preferences in Large Language Models in Molecular Science](https://www.nature.com/articles/s42256-024-00977-6)** |   [![Stars](https://img.shields.io/github/stars/AI-HPC-Research-Team/SLM4Mol?color=yellow&style=social)](https://github.com/AI-HPC-Research-Team/SLM4Mol)
- `Text + Molecule` *`Arxiv 2025`* **[CLASS: Enhancing Cross-Modal Text-Molecule Retrieval Performance and Training Efficiency](https://arxiv.org/abs/2502.11633)** |   
- `Text + Molecule` *`NeurIPS 2025`* **[Mol-LLaMA: Towards General Understanding of Molecules in Large Molecular Language Model](https://arxiv.org/abs/2502.13449)** |   [![Model](https://img.shields.io/badge/Model-5291C8?style=flat&logo=themodelsresource&labelColor=555555)](https://huggingface.co/DongkiKim/Mol-Llama-2-7b-chat) [![Stars](https://img.shields.io/github/stars/DongkiKim95/Mol-LLaMA?color=yellow&style=social)](https://github.com/DongkiKim95/Mol-LLaMA)
- `Text + Molecule` *`Bioinformatics 2025`* **[ChatMol: A Versatile Molecule Designer Based on the Numerically Enhanced Large Language Model](https://arxiv.org/abs/2502.19794)** |   [![Stars](https://img.shields.io/github/stars/ChatMol/ChatMol?color=yellow&style=social)](https://github.com/ChatMol/ChatMol)
- `Text + Molecule` *`EMNLP 2025`* **[MV-CLAM: Multi-View Molecular Interpretation with Cross-Modal Projection via Language Model](https://aclanthology.org/2025.findings-emnlp.1174/)** |   [![Stars](https://img.shields.io/github/stars/sumin124/mv-clam?color=yellow&style=social)](https://github.com/sumin124/mv-clam)
- `Text + Molecule` *`Arxiv 2025`* **[GraphT5: Unified Molecular Graph-Language Modeling via Multi-Modal Cross-Token Attention](https://arxiv.org/pdf/2503.07655)** |   
- `Text + Molecule` *`JBHI 2025`* **[XMolCap: Advancing Molecular Captioning through Multimodal Fusion and Explainable Graph Neural Networks](https://ieeexplore.ieee.org/abstract/document/11012653)** |   [![Stars](https://img.shields.io/github/stars/cbbl-skku-org/XMolCap?color=yellow&style=social)](https://github.com/cbbl-skku-org/XMolCap)
- `Text + Molecule` *`JCIM 2025`* **[(ChatChemTS) Large language models open new way of AI-assisted molecule design for chemists](https://ieeexplore.ieee.org/abstract/document/11012653)** |   [![Stars](https://img.shields.io/github/stars/molecule-generator-collection/ChatChemTS?color=yellow&style=social)](https://github.com/molecule-generator-collection/ChatChemTS)
- `Text + Molecule` *`JCIM 2025`* **[(LLM-MPP) Effective and Explainable Molecular Property Prediction by Chain-of-Thought Enabled Large Language Models and Multi-Modal Molecular Information Fusion](https://pubs.acs.org/doi/full/10.1021/acs.jcim.5c00577)** |   [![Stars](https://img.shields.io/github/stars/jinchang1223/LLM-MPP?color=yellow&style=social)](https://github.com/jinchang1223/LLM-MPP)
- `Text + Molecule` *`AAAI 2025`* **[Graph2Token: Make LLMs Understand Molecule Graphs](https://doi.org/10.1609/aaai.v39i20.35422)** |   [![Stars](https://img.shields.io/github/stars/ZeLeBron/Graph2Token?color=yellow&style=social)](https://github.com/ZeLeBron/Graph2Token)
- `Text + Molecule` *`AAAI 2025`* **[ExDDI: Explaining Drug-Drug Interaction Predictions with Natural Language](https://ojs.aaai.org/index.php/AAAI/article/view/34709)** |   [![Stars](https://img.shields.io/github/stars/ZhaoyueSun/ExDDI?color=yellow&style=social)](https://github.com/ZhaoyueSun/ExDDI)
- `Text + Molecule` *`AAAI 2025`* **[ChemVLM: Exploring the Power of Multimodal Large Language Models in Chemistry Area](https://ojs.aaai.org/index.php/AAAI/article/view/32020)** |   [![Model](https://img.shields.io/badge/Model-5291C8?style=flat&logo=themodelsresource&labelColor=555555)](https://huggingface.co/AI4Chem/ChemVLM-26B-1-2) [![Stars](https://img.shields.io/github/stars/lijunxian111/ChemVlm?color=yellow&style=social)](https://github.com/lijunxian111/ChemVlm)
- `Text + Molecule` *`ICJAI 2025`* **[ChemDual: Enhancing Chemical Reaction and Retrosynthesis Prediction with Large Language Model and Dual-task Learning](https://arxiv.org/abs/2505.02639)** |   [![Stars](https://img.shields.io/github/stars/JacklinGroup/ChemDual?color=yellow&style=social)](https://github.com/JacklinGroup/ChemDual)
- `Text + Molecule` *`ACL 2025`* **[GeLLM¬≥O: Generalizing Large Language Models for Multi-property Molecule Optimization](https://aclanthology.org/anthology-files/anthology-files/pdf/acl/2025.acl-long.1225.pdf)** |   [![Model](https://img.shields.io/badge/Model-5291C8?style=flat&logo=themodelsresource&labelColor=555555)](https://huggingface.co/NingLab/GeLLMO-P6-Mistral) [![Stars](https://img.shields.io/github/stars/ninglab/GeLLMO?color=yellow&style=social)](https://github.com/ninglab/GeLLMO)
- `Text + Molecule` *`ACL 2025`* **[Less for More: Enhanced Feedback-aligned Mixed LLMs for Molecule Caption Generation and Fine-Grained NLI Evaluation](https://aclanthology.org/2025.acl-long.144/)** |   
- `Text + Molecule` *`EMNLP 2025`* **[(GeLLMO-C) Large Language Models for Controllable Multi-property Multi-objective Molecule Optimization](https://aclanthology.org/2025.findings-emnlp.1145/)** |   [![Model](https://img.shields.io/badge/Model-5291C8?style=flat&logo=themodelsresource&labelColor=555555)](https://huggingface.co/collections/NingLab/controllable-gellmo) [![Stars](https://img.shields.io/github/stars/ninglab/GeLLMO-C?color=yellow&style=social)](https://github.com/ninglab/GeLLMO-C)
- `Text + Molecule` *`AAAI 2025`* **[ReactGPT: Understanding of Chemical Reactions via In-Context Tuning](https://ojs.aaai.org/index.php/AAAI/article/view/31983)** |   
- `Text + Molecule` *`AAAI 2025`* **[nach0-pc: Multi-task Language Model with Molecular Point Cloud Encoder](https://doi.org/10.1609/aaai.v39i23.34613)** |   
- `Text + Molecule` *`Arxiv 2025`* **[mCLM: A Function-Infused and Synthesis-Friendly Modular Chemical Language Model](https://arxiv.org/abs/2505.12565)** |   [![Model](https://img.shields.io/badge/Model-5291C8?style=flat&logo=themodelsresource&labelColor=555555)](https://huggingface.co/collections/language-plus-molecules/mclm) 
- `Text + Molecule` *`Arxiv 2025`* **[ChemMLLM: Chemical Multimodal Large Language Model](https://arxiv.org/abs/2505.16326)** |   [![Stars](https://img.shields.io/github/stars/bbsbz/ChemMLLM?color=yellow&style=social)](https://github.com/bbsbz/ChemMLLM)
- `Text + Molecule` *`Arxiv 2025`* **[(CLEANMOL) Improving Chemical Understanding of LLMs via SMILES Parsing](https://arxiv.org/abs/2505.16340)** |   
- `Text + Molecule` *`NeurIPS 2025`* **[ModuLM: Enabling Modular and Multimodal Molecular Relational Learning with Large Language Models](https://arxiv.org/html/2506.00880v1)** |   [![Model](https://img.shields.io/badge/Model-5291C8?style=flat&logo=themodelsresource&labelColor=555555)](https://anonymous.4open.science/r/ModuLM/README.md) 
- `Text + Molecule` *`arXiv 2025`* **[(ToDi) TextOmics-Guided Diffusion for Hit-like Molecular Generation](https://arxiv.org/abs/2507.09982)** |   
- `Text + Molecule` *`Arxiv 2025`* **[ChemDFM-R: An Chemical Reasoner LLM Enhanced with Atomized Chemical Knowledge](https://arxiv.org/pdf/2507.21990v2)** |   
- `Text + Molecule` *`PAKDD 2025`* **[Dual Learning Between Molecules and Natural Language](https://link.springer.com/chapter/10.1007/978-981-96-8173-0_31)** |   
- `Text + Molecule` *`MM 2025`* **[CROP: Integrating Topological and Spatial Structures via Cross-View Prefixes for Molecular LLMs](https://arxiv.org/pdf/2508.06917)** |   
- `Text + Molecule` *`Arxiv 2025`* **[Mol-R1: Towards Explicit Long-CoT Reasoning in Molecule Discovery](https://arxiv.org/abs/2508.08401)** |   
- `Text + Molecule` *`Arxiv 2025`* **[AttriLens-Mol: Attribute Guided Reinforcement Learning for Molecular Property Prediction with Large Language Models](https://arxiv.org/abs/2508.04748)** |   [![Stars](https://img.shields.io/github/stars/szu-tera/AttriLens-Mol?color=yellow&style=social)](https://github.com/szu-tera/AttriLens-Mol)
- `Text + Molecule` *`Bioinformatics 2025`* **[MolPrompt: improving multi-modal molecular pre-training with knowledge prompts](https://academic.oup.com/bioinformatics/article/41/9/btaf466/8240326?login=false)** |   [![Stars](https://img.shields.io/github/stars/catly/MolPrompt?color=yellow&style=social)](https://github.com/catly/MolPrompt)
- `Text + Molecule` *`ENMLP 2025`* **[(CAMT5) Training Text-to-Molecule Models with Context-Aware Tokenization](https://aclanthology.org/2025.findings-emnlp.1221/)** |   [![Stars](https://img.shields.io/github/stars/Songhyeontae/CAMT5?color=yellow&style=social)](https://github.com/Songhyeontae/CAMT5)
- `Text + Molecule` *`Arxiv 2025`* **[Enhancing Molecular Property Prediction with Knowledge from Large Language Models](https://arxiv.org/abs/2509.20664)** |   
- `Text + Molecule` *`Knosys 2025`* **[(MolFinePrompt) Fine-grained multimodal molecular pretraining via prompt learning](https://doi.org/10.1016/j.knosys.2025.114381)** |   [![Stars](https://img.shields.io/github/stars/yzf-code/KnowMol?color=yellow&style=social)](https://github.com/catly/MolFinePrompt)
- `Text + Molecule` *`Arxiv 2025`* **[(MPPReasoner) Reasoning-Enhanced Large Language Models for Molecular Property Prediction](https://arxiv.org/abs/2510.10248)** |   [![Model](https://img.shields.io/badge/Model-5291C8?style=flat&logo=themodelsresource&labelColor=555555)](https://anonymous.4open.science/r/MPPReasoner-12687/README.md) [![Stars](https://img.shields.io/github/stars/Jesse-zjx/MPPReasoner?color=yellow&style=social)](https://github.com/Jesse-zjx/MPPReasoner)
- `Text + Molecule` *`Arxiv 2025`* **[(MECo) Coder as Editor: Code-Driven Interpretable Molecule Optimization](https://arxiv.org/pdf/2510.14455)** |   
- `Text + Molecule` *`Arxiv 2025`* **[Chem-R: Learning to Reason as a Chemist](https://arxiv.org/abs/2510.16880)** |   [![Stars](https://img.shields.io/github/stars/davidweidawang/Chem-R?color=yellow&style=social)](https://github.com/davidweidawang/Chem-R)
- `Text + Molecule` *`Arxiv 2025`* **[KnowMol: Advancing Molecular Large Language Models with Multi-Level Chemical Knowledge](https://arxiv.org/abs/2510.19484)** |   [![Model](https://img.shields.io/badge/Model-5291C8?style=flat&logo=themodelsresource&labelColor=555555)](https://hf.co/datasets/yzf1102/KnowMol-100K) [![Stars](https://img.shields.io/github/stars/yzf-code/KnowMol?color=yellow&style=social)](https://github.com/yzf-code/KnowMol)
- `Text + Molecule` *`IEEE Trans Comput Soc Syst 2025`* **[(Mol-LLM) Incorporating Molecular Knowledge in Large Language Models via Multimodal Modeling](https://ieeexplore.ieee.org/abstract/document/10838383)** |   
- `Text + Molecule` *`MM 2025`* **[DeepMolTex: Deep Alignment of Molecular Graphs with Large Language Models via Mixture of Modality Experts](https://dl.acm.org/doi/abs/10.1145/3746027.3755875)** |   
- `Text + Molecule` *`Neurocomputing 2025`* **[Mol-L2: Transferring text knowledge with frozen language models for molecular representation learning](https://www.sciencedirect.com/science/article/abs/pii/S0925231225015097)** |   
- `Text + Molecule` *`ACL 2025`* **[(MolRAG) Unlocking the Power of LLMs for Molecular Property Prediction](https://aclanthology.org/2025.acl-long.755/)** |   [![Stars](https://img.shields.io/github/stars/AcaciaSin/MolRAG?color=yellow&style=social)](https://github.com/AcaciaSin/MolRAG)
- `Text + Molecule` *`ACL 2025`* **[Boosting LLM's Molecular Structure Elucidation with Knowledge Enhanced Tree Search Reasoning](https://aclanthology.org/2025.acl-long.1100/)** |   [![Stars](https://img.shields.io/github/stars/HICAI-ZJU/K-MSE?color=yellow&style=social)](https://github.com/HICAI-ZJU/K-MSE)
- `Text + Molecule` *`EMNLP 2025`* **[(MolBridge) Bridging the Gap Between Molecule and Textual Descriptions via Substructure-aware Alignment](https://aclanthology.org/2025.emnlp-main.1197/)** |   [![Stars](https://img.shields.io/github/stars/Park-ing-lot/MolBridge?color=yellow&style=social)](https://github.com/Park-ing-lot/MolBridge)
- `Text + Molecule` *`EMNLP 2025`* **[GAMIC: Graph-Aligned Molecular In-context Learning for Molecule Analysis via LLMs](https://aclanthology.org/2025.findings-emnlp.996/)** |   [![Stars](https://img.shields.io/github/stars/aliwister/mol-icl?color=yellow&style=social)](https://github.com/aliwister/mol-icl)
- `Text + Molecule` *`EMNLP 2025`* **[How to Make Large Language Models Generate 100% Valid Molecules?](https://aclanthology.org/2025.emnlp-main.1350/)** |   [![Stars](https://img.shields.io/github/stars/wentao228/SmiSelf?color=yellow&style=social)](https://github.com/wentao228/SmiSelf)
- `Text + Molecule` *`EMNLP 2025`* **[Molecular String Representation Preferences in Pretrained LLMs: A Comparative Study in Zero- & Few-Shot Molecular Property Prediction](https://aclanthology.org/2025.emnlp-main.56/)** |   [![Stars](https://img.shields.io/github/stars/Spongeorge/llm-mpp?color=yellow&style=social)](https://github.com/Spongeorge/llm-mpp)
- `Text + Protein` *`ICLR 2022`* **[OntoProtein: Protein Pretraining With Gene Ontology Embedding](https://openreview.net/pdf?id=yfe1VMYAXa4)** |   [![Model](https://img.shields.io/badge/Model-5291C8?style=flat&logo=themodelsresource&labelColor=555555)](https://huggingface.co/zjunlp/OntoProtein) [![Stars](https://img.shields.io/github/stars/zjunlp/OntoProtein?color=yellow&style=social)](https://github.com/zjunlp/OntoProtein)
- `Text + Protein` *`RECOMB 2022`* **[ProTranslator: Zero-Shot Protein Function Prediction Using Textual Description](https://arxiv.org/abs/2204.10286)** |   [![Stars](https://img.shields.io/github/stars/HanwenXuTHU/ProTranslator?color=yellow&style=social)](https://github.com/HanwenXuTHU/ProTranslator)
- `Text + Protein` *`ICML 2023`* **[ProtST: Multi-Modality Learning of Protein Sequences and Biomedical Texts](https://proceedings.mlr.press/v202/xu23t/xu23t.pdf)** |   [![Stars](https://img.shields.io/github/stars/DeepGraphLearning/ProtST?color=yellow&style=social)](https://github.com/DeepGraphLearning/ProtST)
- `Text + Protein` *`Nature Biotechnology 2023`* **[(ProGen) Large language models generate functional protein sequences across diverse families](https://www.nature.com/articles/s41587-022-01618-2)** |   [![Stars](https://img.shields.io/github/stars/salesforce/progen?color=yellow&style=social)](https://github.com/salesforce/progen)
- `Text + Protein` *`Arxiv 2023`* **[(ProteinDT) A Text-guided Protein Design Framework](https://arxiv.org/pdf/2302.04611.pdf)** |   [![Stars](https://img.shields.io/github/stars/chao1224/ProteinDT?color=yellow&style=social)](https://github.com/chao1224/ProteinDT)
- `Text + Protein` *`TechRxiv 2023`* **[ProteinChat: Towards Achieving ChatGPT-Like Functionalities on Protein 3D Structures](https://www.techrxiv.org/doi/full/10.36227/techrxiv.23120606.v1)** |   [![Stars](https://img.shields.io/github/stars/UCSD-AI4H/proteinchat?color=yellow&style=social)](https://github.com/UCSD-AI4H/proteinchat)
- `Text + Protein` *`ICML 2025 2023`* **[(ProteinChat v2) Multi-Modal Large Language Model Enables Protein Function Prediction](https://www.biorxiv.org/content/10.1101/2024.08.19.608729v1.abstract)** |   [![Model](https://img.shields.io/badge/Model-5291C8?style=flat&logo=themodelsresource&labelColor=555555)](https://drive.google.com/file/d/1H-POt4e5Q5fYF59ZwfSdAJyuQiJ2rtJl/view) [![Stars](https://img.shields.io/github/stars/mignonjia/ProteinChat?color=yellow&style=social)](https://github.com/mignonjia/ProteinChat)
- `Text + Protein` *`AAAI 2024`* **[Prot2Text: Multimodal Protein's Function Generation with GNNs and Transformers](https://arxiv.org/pdf/2307.14367.pdf)** |   [![Stars](https://img.shields.io/github/stars/hadi-abdine/Prot2Text?color=yellow&style=social)](https://github.com/hadi-abdine/Prot2Text)
- `Text + Protein` *`Arxiv 2024`* **[ProtAgents: Protein discovery via large language model multi-agent collaborations combining physics and machine learning](https://arxiv.org/pdf/2402.04268.pdf)** |   [![Stars](https://img.shields.io/github/stars/lamm-mit/ProtAgents?color=yellow&style=social)](https://github.com/lamm-mit/ProtAgents)
- `Text + Protein` *`TAI 2024`* **[ProLLaMA: A Protein Large Language Model for Multi-Task Protein Language Processing](https://ieeexplore.ieee.org/abstract/document/10979347)** |   [![Model](https://img.shields.io/badge/Model-5291C8?style=flat&logo=themodelsresource&labelColor=555555)](https://huggingface.co/GreatCaptainNemo/ProLLaMA) [![Stars](https://img.shields.io/github/stars/Lyu6PosHao/ProLLaMA?color=yellow&style=social)](https://github.com/Lyu6PosHao/ProLLaMA)
- `Text + Protein` *`ACL 2024`* **[ProtLLM: An Interleaved Protein-Language LLM with Protein-as-Word Pre-Training](https://arxiv.org/pdf/2403.07920.pdf)** |   [![Model](https://img.shields.io/badge/Model-5291C8?style=flat&logo=themodelsresource&labelColor=555555)](https://huggingface.co/datasets/ProtLLM/ProtLLM) [![Stars](https://img.shields.io/github/stars/ProtLLM/ProtLLM?color=yellow&style=social)](https://github.com/ProtLLM/ProtLLM)
- `Text + Protein` *`ACL 2024`* **[ProtT3: Protein-to-Text Generation for Text-based Protein Understanding](https://arxiv.org/pdf/2405.12564)** |   [![Model](https://img.shields.io/badge/Model-5291C8?style=flat&logo=themodelsresource&labelColor=555555)](https://osf.io/23azs/overview?view_only=185575515e714f4798499bf06513a730) [![Stars](https://img.shields.io/github/stars/acharkq/ProtT3?color=yellow&style=social)](https://github.com/acharkq/ProtT3)
- `Text + Protein` *`bioRxiv 2024`* **[ProteinCLIP: enhancing protein language models with natural language](https://www.biorxiv.org/content/10.1101/2024.05.14.594226v1.full.pdf)** |   [![Stars](https://img.shields.io/github/stars/wukevin/proteinclip?color=yellow&style=social)](https://github.com/wukevin/proteinclip)
- `Text + Protein` *`Arxiv 2024`* **[ProLLM: Protein Chain-of-Thoughts Enhanced LLM for Protein-Protein Interaction Prediction](https://arxiv.org/pdf/2405.06649)** |   [![Stars](https://img.shields.io/github/stars/MingyuJ666/ProLLM?color=yellow&style=social)](https://github.com/MingyuJ666/ProLLM)
- `Text + Protein` *`KDD 2025`* **[(PAAG) Annotation-guided Protein Design with Multi-Level Domain Alignment](https://arxiv.org/pdf/2404.16866)** |   [![Model](https://img.shields.io/badge/Model-5291C8?style=flat&logo=themodelsresource&labelColor=555555)](https://huggingface.co/ychaohao/PAAG) [![Stars](https://img.shields.io/github/stars/chaohaoyuan/PAAG?color=yellow&style=social)](https://github.com/chaohaoyuan/PAAG)
- `Text + Protein` *`bioRrxiv 2024`* **[(Pinal) Toward De Novo Protein Design from Natural Language](https://www.biorxiv.org/content/10.1101/2024.08.01.606258v1.full.pdf)** |   [![Model](https://img.shields.io/badge/Model-5291C8?style=flat&logo=themodelsresource&labelColor=555555)](https://huggingface.co/westlake-repl/Pinal) [![Stars](https://img.shields.io/github/stars/westlake-repl/Denovo-Pinal?color=yellow&style=social)](https://github.com/westlake-repl/Denovo-Pinal)
- `Text + Protein` *`Arxiv 2024`* **[TourSynbio: A Multi-Modal Large Model and Agent Framework to Bridge Text and Protein Sequences for Protein Engineering](https://www.arxiv.org/pdf/2408.15299)** |   [![Model](https://img.shields.io/badge/Model-5291C8?style=flat&logo=themodelsresource&labelColor=555555)](https://huggingface.co/tsynbio/Toursynbio) [![Stars](https://img.shields.io/github/stars/tsynbio/TourSynbio?color=yellow&style=social)](https://github.com/tsynbio/TourSynbio)
- `Text + Protein` *`KDD 2025`* **[(SEPIT) Structure-Enhanced Protein Instruction Tuning: Towards General-Purpose Protein Understanding with LLMs](https://arxiv.org/abs/2410.03553)** |   [![Model](https://img.shields.io/badge/Model-5291C8?style=flat&logo=themodelsresource&labelColor=555555)](https://huggingface.co/U-rara/SEPIT) [![Stars](https://img.shields.io/github/stars/U-rara/SEPIT?color=yellow&style=social)](https://github.com/U-rara/SEPIT)
- `Text + Protein` *`NeurIPS 2024`* **[(BioM3) Natural Language Prompts Guide the Design of Novel Functional Protein Sequences](https://openreview.net/forum?id=L1MyyRCAjX)** |   [![Model](https://img.shields.io/badge/Model-5291C8?style=flat&logo=themodelsresource&labelColor=555555)](https://huggingface.co/niksapraljak1/BioM3) [![Stars](https://img.shields.io/github/stars/PraljakReps/BioM3?color=yellow&style=social)](https://github.com/PraljakReps/BioM3)
- `Text + Protein` *`NeurIPS 2024`* **[(LLM4ProteinEvolution) Language Models for Text-guided Protein Evolution](https://openreview.net/forum?id=CNla8z0V2p)** |   [![Stars](https://img.shields.io/github/stars/ZhanghanNi/LLM4ProteinEvolution?color=yellow&style=social)](https://github.com/ZhanghanNi/LLM4ProteinEvolution)
- `Text + Protein` *`NeurIPS 2024`* **[MMSite: A Multi-modal Framework for the Identification of Active Sites in Proteins](https://neurips.cc/virtual/2024/poster/94780)** |   [![Model](https://img.shields.io/badge/Model-5291C8?style=flat&logo=themodelsresource&labelColor=555555)](https://zenodo.org/records/14599105) [![Stars](https://img.shields.io/github/stars/Gift-OYS/MMSite?color=yellow&style=social)](https://github.com/Gift-OYS/MMSite)
- `Text + Protein` *`NeurIPS 2024`* **[MutaPLM: Protein Language Modeling for Mutation Explanation and Engineering](https://neurips.cc/virtual/2024/poster/92987)** |   [![Model](https://img.shields.io/badge/Model-5291C8?style=flat&logo=themodelsresource&labelColor=555555)](https://huggingface.co/PharMolix/MutaPLM) [![Stars](https://img.shields.io/github/stars/PharMolix/MutaPLM?color=yellow&style=social)](https://github.com/PharMolix/MutaPLM)
- `Text + Protein` *`Arxiv 2024`* **[ProtDAT: A Unified Framework for Protein Sequence Design from Any Protein Text Description](https://arxiv.org/abs/2412.04069)** |   [![Model](https://img.shields.io/badge/Model-5291C8?style=flat&logo=themodelsresource&labelColor=555555)](https://zenodo.org/records/14264096) [![Stars](https://img.shields.io/github/stars/GXY0116/ProtDAT?color=yellow&style=social)](https://github.com/GXY0116/ProtDAT)
- `Text + Protein` *`Arxiv 2024`* **[EvoLlama: Enhancing LLMs' Understanding of Proteins via Multimodal Structure and Sequence Representations](https://arxiv.org/pdf/2412.11618)** |   [![Model](https://img.shields.io/badge/Model-5291C8?style=flat&logo=themodelsresource&labelColor=555555)](https://huggingface.co/nwliu/EvoLlama) [![Stars](https://img.shields.io/github/stars/sornkL/EvoLlama?color=yellow&style=social)](https://github.com/sornkL/EvoLlama)
- `Text + Protein` *`Bioinformatics 2024`* **[FAPM: functional annotation of proteins using multimodal models beyond structural modeling](https://academic.oup.com/bioinformatics/article/40/12/btae680/7900294)** |   [![Model](https://img.shields.io/badge/Model-5291C8?style=flat&logo=themodelsresource&labelColor=555555)](https://huggingface.co/wenkai/FAPM/tree/main/model) [![Stars](https://img.shields.io/github/stars/xiangwenkai/FAPM?color=yellow&style=social)](https://github.com/xiangwenkai/FAPM)
- `Text + Protein` *`bioRxiv 2024`* **[ProCyon: A multimodel foundation model for protein phenotypes](https://www.biorxiv.org/content/10.1101/2024.12.10.627665v1)** |   [![Model](https://img.shields.io/badge/Model-5291C8?style=flat&logo=themodelsresource&labelColor=555555)](https://huggingface.co/mims-harvard/ProCyon-Full) [![Stars](https://img.shields.io/github/stars/mims-harvard/ProCyon?color=yellow&style=social)](https://github.com/mims-harvard/ProCyon)
- `Text + Protein` *`bioRxiv 2025`* **[(Evolla) Decoding the Molecular Language of Proteins with Evolla](https://www.biorxiv.org/content/10.1101/2025.01.05.630192v1)** |   [![Model](https://img.shields.io/badge/Model-5291C8?style=flat&logo=themodelsresource&labelColor=555555)](https://huggingface.co/westlake-repl/Evolla-10B) [![Stars](https://img.shields.io/github/stars/westlake-repl/Evolla?color=yellow&style=social)](https://github.com/westlake-repl/Evolla)
- `Text + Protein` *`ICLR 2025`* **[ProteinGPT: Multimodal LLM for Protein Property Prediction and Structure Understanding](https://openreview.net/pdf?id=qWlqfjGVWX)** |   [![Stars](https://img.shields.io/github/stars/OviaLabs/ProteinGPT?color=yellow&style=social)](https://github.com/OviaLabs/ProteinGPT)
- `Text + Protein` *`ICLR 2025`* **[(MP4) A generalized protein design ML model enables generation of functional de novo proteins](https://openreview.net/forum?id=H7KaafflRG)** |   
- `Text + Protein` *`AAAI 2025`* **[Protclip: Function-informed protein multi-modal learning](https://ojs.aaai.org/index.php/AAAI/article/view/34456)** |   [![Model](https://img.shields.io/badge/Model-5291C8?style=flat&logo=themodelsresource&labelColor=555555)](https://zenodo.org/records/15245588) [![Stars](https://img.shields.io/github/stars/diaoshaoyou/ProtCLIP?color=yellow&style=social)](https://github.com/diaoshaoyou/ProtCLIP)
- `Text + Protein` *`AAAI 2025`* **[(CtrlProt) Controllable Protein Sequence Generation with LLM Preference Optimization Authors](https://ojs.aaai.org/index.php/AAAI/article/view/32030)** |   [![Stars](https://img.shields.io/github/stars/nju-websoft/CtrlProt?color=yellow&style=social)](https://github.com/nju-websoft/CtrlProt)
- `Text + Protein` *`NAACL 2025`* **[Protein2Text: Resampling Mechanism to Translate Protein Sequences into Human-Interpretable Text](https://aclanthology.org/2025.naacl-industry.68/)** |   [![Model](https://img.shields.io/badge/Model-5291C8?style=flat&logo=themodelsresource&labelColor=555555)](https://huggingface.co/tumorailab/protein2text-llama3.1-8B-instruct-esm2-650M) [![Stars](https://img.shields.io/github/stars/alaaj27/Protein2Text?color=yellow&style=social)](https://github.com/alaaj27/Protein2Text)
- `Text + Protein` *`EMNLP 2025`* **[(RAPM): Rethinking Text-based Protein Understanding: Retrieval or LLM?](https://aclanthology.org/2025.emnlp-main.1211/)** |   [![Stars](https://img.shields.io/github/stars/IDEA-XL/RAPM?color=yellow&style=social)](https://github.com/IDEA-XL/RAPM)
- `Text + Protein` *`NeurIPS 2025`* **[Prot2Text-V2: Protein Function Prediction with Multimodal Contrastive Alignment](https://arxiv.org/abs/2505.11194)** |   [![Model](https://img.shields.io/badge/Model-5291C8?style=flat&logo=themodelsresource&labelColor=555555)](https://huggingface.co/xiao-fei/Prot2Text-V2-11B-Instruct-hf) [![Stars](https://img.shields.io/github/stars/colinfx/prot2text-v2?color=yellow&style=social)](https://github.com/colinfx/prot2text-v2)
- `Text + Protein` *`NeurIPS 2025`* **[(ProDVa) Protein Design with Dynamic Protein Vocabulary](https://arxiv.org/abs/2505.18966)** |   [![Model](https://img.shields.io/badge/Model-5291C8?style=flat&logo=themodelsresource&labelColor=555555)](https://huggingface.co/collections/nwliu/prodva) [![Stars](https://img.shields.io/github/stars/sornkL/ProDVa?color=yellow&style=social)](https://github.com/sornkL/ProDVa)
- `Text + Protein` *`ICML 2025`* **[ProteinAligner: A Tri-Modal Contrastive Learning Framework for Protein Representation Learning](https://openreview.net/forum?id=LjlJBnmZ0M)** |   [![Stars](https://img.shields.io/github/stars/Alexiland/ProteinAligner?color=yellow&style=social)](https://github.com/Alexiland/ProteinAligner)
- `Text + Protein` *`JCIM 2025`* **[Prottex: Structure-in-context reasoning and editing of proteins with large language models](https://pubs.acs.org/doi/abs/10.1021/acs.jcim.5c00585)** |   [![Model](https://img.shields.io/badge/Model-5291C8?style=flat&logo=themodelsresource&labelColor=555555)](https://huggingface.co/mzcwd/ProtTeX) [![Stars](https://img.shields.io/github/stars/mzc2113391/ProtTeX?color=yellow&style=social)](https://github.com/mzc2113391/ProtTeX)
- `Text + Protein` *`Bioinformatics 2025`* **[Prot2Chat: protein large language model with early fusion of text, sequence, and structure](https://academic.oup.com/bioinformatics/article/41/8/btaf396/8215464)** |   [![Model](https://img.shields.io/badge/Model-5291C8?style=flat&logo=themodelsresource&labelColor=555555)](https://huggingface.co/zcw51699/prot2chat/tree/main/zcw51699/prot2chat) [![Stars](https://img.shields.io/github/stars/wangzc1233/Prot2Chat?color=yellow&style=social)](https://github.com/wangzc1233/Prot2Chat)
- `Text + Protein` *`Nature Biotechnology 2025`* **[ProTrek: Navigating the Protein Universe through Tri-Modal Contrastive Learning](https://www.nature.com/articles/s41587-025-02836-0)** |   [![Model](https://img.shields.io/badge/Model-5291C8?style=flat&logo=themodelsresource&labelColor=555555)](https://huggingface.co/westlake-repl/ProTrek_650M_UniRef50) [![Stars](https://img.shields.io/github/stars/westlake-repl/ProTrek?color=yellow&style=social)](https://github.com/westlake-repl/ProTrek)
- `Text + Protein` *`ACL 2025`* **[Enhancing Safe and Controllable Protein Generation via Knowledge Preference Optimization](https://aclanthology.org/2025.acl-long.616/)** |   [![Stars](https://img.shields.io/github/stars/HICAI-ZJU/KPO?color=yellow&style=social)](https://github.com/HICAI-ZJU/KPO)
- `Text + Protein` *`ACL 2025`* **[(LLaPA) Large Language and Protein Assistant for Protein-Protein Interactions Prediction](https://aclanthology.org/2025.acl-long.554/)** |   [![Stars](https://img.shields.io/github/stars/HHW-zhou/LLAPA?color=yellow&style=social)](https://github.com/HHW-zhou/LLAPA)
- `Text + Protein` *`Arxiv 2025`* **[Protein as a Section Language for LLMs](https://arxiv.org/pdf/2510.11188)** |   
- `Text + Protein` *`Arxiv 2025`* **[CMADiff: Cross-Modal Aligned Diffusion for Controllable Protein Generation](https://arxiv.org/pdf/2503.21450)** |
- `Text + Protein` *`ICLR Submission 2026`* **[Caduceus: MoE-enhanced Foundation Models Unifying Biological and Natural Language](https://openreview.net/forum?id=NNRQW01Xuh)** |   
- `Text + BioMulti` *`Arxiv 2022`* **[Galactica: A Large Language Model for Science](https://galactica.org/static/paper.pdf)** |   [![Model](https://img.shields.io/badge/Model-5291C8?style=flat&logo=themodelsresource&labelColor=555555)](https://huggingface.co/models?other=galactica) [![Stars](https://img.shields.io/github/stars/paperswithcode/galai?color=yellow&style=social)](https://github.com/paperswithcode/galai)
- `Text + BioMulti` *`EMNLP 2023`* **[BioT5: Enriching Cross-modal Integration in Biology with Chemical Knowledge and Natural Language Associations](https://aclanthology.org/2023.emnlp-main.70.pdf)** |   [![Model](https://img.shields.io/badge/Model-5291C8?style=flat&logo=themodelsresource&labelColor=555555)](https://huggingface.co/QizhiPei/biot5-base) [![Stars](https://img.shields.io/github/stars/QizhiPei/BioT5?color=yellow&style=social)](https://github.com/QizhiPei/BioT5)
- `Text + BioMulti` *`Arxiv 2023`* **[DARWIN Series: Domain Specific Large Language Models for Natural Science](https://arxiv.org/pdf/2308.13565.pdf)** |   [![Model](https://img.shields.io/badge/Model-5291C8?style=flat&logo=themodelsresource&labelColor=555555)](https://github.com/MasterAI-EAM/Darwin) [![Stars](https://img.shields.io/github/stars/MasterAI-EAM/Darwin?color=yellow&style=social)](https://github.com/MasterAI-EAM/Darwin)
- `Text + BioMulti` *`Arxiv 2023`* **[BioMedGPT: Open Multimodal Generative Pre-trained Transformer for BioMedicine](https://arxiv.org/pdf/2308.09442.pdf)** |   [![Model](https://img.shields.io/badge/Model-5291C8?style=flat&logo=themodelsresource&labelColor=555555)](https://pan.baidu.com/share/init?surl=iAMBkuoZnNAylhopP5OgEg\&pwd=7a6b) [![Stars](https://img.shields.io/github/stars/PharMolix/OpenBioMed?color=yellow&style=social)](https://github.com/PharMolix/OpenBioMed)
- `Text + BioMulti` *`Arxiv 2023`* **[(StructChem) Structured Chemistry Reasoning with Large Language Models](https://arxiv.org/pdf/2311.09656.pdf)** |   [![Stars](https://img.shields.io/github/stars/ozyyshr/StructChem?color=yellow&style=social)](https://github.com/ozyyshr/StructChem?tab=readme-ov-file)
- `Text + BioMulti` *`Nature Communications 2023`* **[(BioTranslator) Multilingual translation for zero-shot biomedical classification using BioTranslator](https://www.nature.com/articles/s41467-023-36476-2.pdf)** |   [![Model](https://img.shields.io/badge/Model-5291C8?style=flat&logo=themodelsresource&labelColor=555555)](https://figshare.com/articles/dataset/Protein_Pathway_data_tar/20120447) [![Stars](https://img.shields.io/github/stars/HanwenXuTHU/BioTranslatorProject?color=yellow&style=social)](https://github.com/HanwenXuTHU/BioTranslatorProject)
- `Text + BioMulti` *`ICLR 2024`* **[Mol-Instructions: A Large-Scale Biomolecular Instruction Dataset for Large Language Models](https://openreview.net/pdf?id=Tlsdsb6l9n)** |   [![Model](https://img.shields.io/badge/Model-5291C8?style=flat&logo=themodelsresource&labelColor=555555)](https://huggingface.co/zjunlp) [![Stars](https://img.shields.io/github/stars/zjunlp/Mol-Instructions?color=yellow&style=social)](https://github.com/zjunlp/Mol-Instructions)
- `Text + BioMulti` *`ICLR 2024`* **[(ChatDrug) ChatGPT-powered Conversational Drug Editing Using Retrieval and Domain Feedback](https://openreview.net/pdf?id=yRrPfKyJQ2)** |   [![Stars](https://img.shields.io/github/stars/chao1224/ChatDrug?color=yellow&style=social)](https://github.com/chao1224/ChatDrug)
- `Text + BioMulti` *`ICLR 2024`* **[BioBridge: Bridging Biomedical Foundation Models via Knowledge Graphs](https://openreview.net/pdf?id=jJCeMiwHdH)** |   [![Model](https://img.shields.io/badge/Model-5291C8?style=flat&logo=themodelsresource&labelColor=555555)](https://github.com/RyanWangZf/BioBridge/tree/main/checkpoints) [![Stars](https://img.shields.io/github/stars/RyanWangZf/BioBridge?color=yellow&style=social)](https://github.com/RyanWangZf/BioBridge)
- `Text + BioMulti` *`AAAI 2024`* **[(KEDD) Towards Unified AI Drug Discovery with Multiple Knowledge Modalities](https://arxiv.org/pdf/2305.01523.pdf)** |   
- `Text + BioMulti` *`AAAI 2024`* **[ÔºàOtter Knowledge) Knowledge Enhanced Representation Learning for Drug Discovery](https://ojs.aaai.org/index.php/AAAI/article/view/28924)** |   [![Model](https://img.shields.io/badge/Model-5291C8?style=flat&logo=themodelsresource&labelColor=555555)](https://huggingface.co/ibm) [![Stars](https://img.shields.io/github/stars/IBM/otter-knowledge?color=yellow&style=social)](https://github.com/IBM/otter-knowledge)
- `Text + BioMulti` *`Arxiv 2024`* **[ChatCell: Facilitating Single-Cell Analysis with Natural Language](https://arxiv.org/pdf/2402.08303v2.pdf)** |   [![Model](https://img.shields.io/badge/Model-5291C8?style=flat&logo=themodelsresource&labelColor=555555)](https://huggingface.co/zjunlp) [![Stars](https://img.shields.io/github/stars/zjunlp/ChatCell?color=yellow&style=social)](https://github.com/zjunlp/ChatCell)
- `Text + BioMulti` *`ICML 2024`* **[LangCell: Language-Cell Pre-training for Cell Identity Understanding](https://arxiv.org/pdf/2405.06708v2)** |   [![Model](https://img.shields.io/badge/Model-5291C8?style=flat&logo=themodelsresource&labelColor=555555)](https://drive.google.com/drive/folders/1cuhVG9v0YoAnjW-t_WMpQQguajumCBTp) [![Stars](https://img.shields.io/github/stars/PharMolix/LangCell?color=yellow&style=social)](https://github.com/PharMolix/LangCell)
- `Text + BioMulti` *`ACL 2024`* **[BioT5+: Towards Generalized Biological Understanding with IUPAC Integration and Multi-task Tuning](https://arxiv.org/pdf/2402.17810.pdf)** |   [![Model](https://img.shields.io/badge/Model-5291C8?style=flat&logo=themodelsresource&labelColor=555555)](https://github.com/QizhiPei/BioT5) [![Stars](https://img.shields.io/github/stars/QizhiPei/BioT5?color=yellow&style=social)](https://github.com/QizhiPei/BioT5)
- `Text + BioMulti` *`Arxiv 2024`* **[MolBind: Multimodal Alignment of Language, Molecules, and Proteins](https://arxiv.org/pdf/2403.08167.pdf)** |   [![Stars](https://img.shields.io/github/stars/tengxiao1/MolBind?color=yellow&style=social)](https://github.com/tengxiao1/MolBind)
- `Text + BioMulti` *`Arxiv 2024`* **[Uni-SMART: Universal Science Multimodal Analysis and Research Transformer](https://arxiv.org/pdf/2403.10301.pdf)** |   
- `Text + BioMulti` *`ICML 2024`* **[Tag-LLM: Repurposing General-Purpose LLMs for Specialized Domains](https://arxiv.org/pdf/2402.05140.pdf)** |   [![Stars](https://img.shields.io/github/stars/sjunhongshen/Tag-LLM?color=yellow&style=social)](https://github.com/sjunhongshen/Tag-LLM)
- `Text + BioMulti` *`Arxiv 2024`* **[An Evaluation of Large Language Models in Bioinformatics Research](https://arxiv.org/pdf/2402.13714.pdf)** |   
- `Text + BioMulti` *`bioRxiv 2024`* **[SciMind: A Multimodal Mixture-of-Experts Model for Advancing Pharmaceutical Sciences](https://www.biorxiv.org/content/10.1101/2024.07.16.603812v1.full.pdf)** |   
- `Text + BioMulti` *`NeurIPS 2024`* **[SciDFM: A Large Language Model with Mixture-of-Experts for Science](https://arxiv.org/abs/2409.18412)** |   [![Model](https://img.shields.io/badge/Model-5291C8?style=flat&logo=themodelsresource&labelColor=555555)](https://huggingface.co/OpenDFM/SciDFM-MoE-A5.6B-v1.0) 
- `Text + BioMulti` *`Sci China Inf Sci 2024`* **[ChemDFM-X: Towards Large Multimodal Model for Chemistry](https://arxiv.org/abs/2409.13194)** |   [![Model](https://img.shields.io/badge/Model-5291C8?style=flat&logo=themodelsresource&labelColor=555555)](https://huggingface.co/OpenDFM/ChemDFM-X-v1.0-13B) [![Stars](https://img.shields.io/github/stars/OpenDFM/ChemDFM-X?color=yellow&style=social)](https://github.com/OpenDFM/ChemDFM-X)
- `Text + BioMulti` *`Arxiv 2025`* **[(NatureLM) Nature Language Model: Deciphering the Language of Nature for Scientific Discovery](https://arxiv.org/abs/2502.07527)** |   [![Model](https://img.shields.io/badge/Model-5291C8?style=flat&logo=themodelsresource&labelColor=555555)](https://huggingface.co/collections/microsoft/naturelm) [![Stars](https://img.shields.io/github/stars/microsoft/SFM?color=yellow&style=social)](https://github.com/microsoft/SFM)
- `Text + BioMulti` *`Cell Rep Phys Sci 2024`* **[(ChemDFM) Developing ChemDFM as a large language foundation model for chemistry](https://www.cell.com/cell-reports-physical-science/fulltext/S2666-3864(25)** |   [![Model](https://img.shields.io/badge/Model-5291C8?style=flat&logo=themodelsresource&labelColor=555555)](https://huggingface.co/OpenDFM/ChemDFM-v1.0-13B) [![Stars](https://img.shields.io/github/stars/OpenDFM/ChemDFM?color=yellow&style=social)](https://github.com/OpenDFM/ChemDFM)
- `Text + BioMulti` *`Information Fusion 2025`* **[(KFPPIMI) Improving protein‚Äìprotein interaction modulator predictions via knowledge-fused language models](https://www.sciencedirect.com/science/article/abs/pii/S1566253525003008?via%3Dihub)** |   [![Stars](https://img.shields.io/github/stars/1zzt/KFPPIMI?color=yellow&style=social)](https://github.com/1zzt/KFPPIMI)
- `Text + BioMulti` *`Arxiv 2025`* **[STELLA: Towards Protein Function Prediction with Multimodal LLMs Integrating Sequence-Structure Representations](https://arxiv.org/abs/2506.03800)** |   [![Model](https://img.shields.io/badge/Model-5291C8?style=flat&logo=themodelsresource&labelColor=555555)](https://anonymous.4open.science/r/STELLA-DF00/README.md) 
- `Text + BioMulti` *`Arxiv 2025`* **[(CAFT) Improving Large Language Models with Concept-Aware Fine-Tuning](https://arxiv.org/pdf/2506.07833)** |   [![Stars](https://img.shields.io/github/stars/michaelchen-lab/caft-llm?color=yellow&style=social)](https://github.com/michaelchen-lab/caft-llm)
- `Text + BioMulti` *`Arxiv 2025`* **[InstructPro: Natural Language Guided Ligand-Binding Protein Design](https://arxiv.org/abs/2506.09332)** |   
- `Text + BioMulti` *`NMI 2025`* **[(InstructBioMol) Advancing biomolecular understanding and design following human instructions](https://www.nature.com/articles/s42256-025-01064-0)** |   [![Stars](https://img.shields.io/github/stars/HICAI-ZJU/InstructBioMol?color=yellow&style=social)](https://github.com/HICAI-ZJU/InstructBioMol)
- `Text + BioMulti` *`bioRxiv 2025`* **[DrugLM: A Unified Framework to Enhance Drug-Target Interaction Predictions by Incorporating Textual Embeddings via Language Models](https://www.biorxiv.org/content/10.1101/2025.07.09.657250v1.abstract)** |   [![Stars](https://img.shields.io/github/stars/HICAI-ZJU/InstructBioMol?color=yellow&style=social)](https://github.com/HICAI-ZJU/InstructBioMol)
- `Text + BioMulti` *`Arxiv 2025`* **[Intern-S1: A Scientific Multimodal Foundation Model](https://arxiv.org/abs/2508.15763)** |   [![Model](https://img.shields.io/badge/Model-5291C8?style=flat&logo=themodelsresource&labelColor=555555)](https://huggingface.co/internlm/Intern-S1) 
- `Text + BioMulti` *`Arxiv 2025`* **[Chem3DLLM: 3D Multimodal Large Language Models for Chemistry](https://arxiv.org/abs/2508.10696)** |   
- `Text + BioMulti` *`Arxiv 2025`* **[SciReasoner: Laying the Scientific Reasoning Ground Across Disciplines](https://arxiv.org/abs/2509.21320)** |   [![Stars](https://img.shields.io/github/stars/open-sciencelab/SciReason?color=yellow&style=social)](https://github.com/open-sciencelab/SciReason)
- `Text + BioMulti` *`Arxiv 2025`* **[MolChord: Structure-Sequence Alignment for Protein-Guided Drug Design](https://arxiv.org/abs/2510.27671)** |   

## Datasets & Benchmarks
- `Pre-training`- `Text` [PubMed](https://pubmed.ncbi.nlm.nih.gov/download)
- `Pre-training`- `Text` [bioRxiv](https://huggingface.co/datasets/mteb/raw_biorxiv)
- `Pre-training`- `Text` [MedRxiv](https://www.medrxiv.org/tdm)
- `Pre-training`- `Text` [S2ORC](https://github.com/allenai/s2orc)
- `Pre-training`- `Text` [MIMIC](https://physionet.org/content/mimiciii/1.4)
- `Pre-training`- `Text` [UF Health](https://idr.ufhealth.org)
- `Pre-training`- `Text` [Elsevier Corpus](https://elsevier.digitalcommonsdata.com/datasets/zm33cdndxs/3)
- `Pre-training`- `Text` [Eurpoe PMC](https://europepmc.org/downloads)
- `Pre-training`- `Text` [LibreText](https://chem.libretexts.org)
- `Pre-training`- `Text` [NLM literature archive](https://ftp.ncbi.nlm.nih.gov/pub/litarch/)
- `Pre-training`- `Text` [GAP-Replay](-)
- `Pre-training`- `Molecule` [ZINC](https://zinc15.docking.org)
- `Pre-training`- `Protein` [UniProt](https://www.uniprot.org)
- `Pre-training`- `Molecule, Bioassay` [ChEMBL](https://www.ebi.ac.uk/chembl)
- `Pre-training`- `Molecule, Bioassay` [GIMLET](https://github.com/zhao-ht/GIMLET)
- `Pre-training`- `Text, Molecule` [mCLM](https://huggingface.co/datasets/language-plus-molecules/mCLM_Pretrain_All)
- `Pre-training`- `Text, Molecule` [PEIT-GEN](https://pan.baidu.com/s/1VcFvrVHmjBZpL2L_QWt9TQ?pwd=vvts)
- `Pre-training`- `Text, Molecule, IUPAC, etc` [PubChem](https://ftp.ncbi.nlm.nih.gov/pubchem)
- `Pre-training`- `Text, Protein` [InterPT](https://huggingface.co/datasets/ProtLLM/ProtLLM)
- `Pre-training`- `Text, Protein, etc` [STRING](https://string-db.org)
- `Pre-training`- `Text, Molecule` [MolTextNet](https://huggingface.co/datasets/liuganghuggingface/moltextnet)
- `Pre-training`- `Text, Protein` [ProtAnnotation](https://huggingface.co/datasets/ychaohao/ProtAnnotation)
- `Pre-training`- `Text, Protein` [ProteinKG25](https://osf.io/23azs/files?view_only=185575515e714f4798499bf06513a730)
- `Pre-training`- `Text, Protein` [ProtAnno](https://zenodo.org/records/15245588)
- `Pre-training`- `Text, Protein` [ProtDescribe](https://drive.google.com/file/d/1iMgPyiIzpvXdKiNsXnRKn2YpmP92Xyub/view)
- `Pre-training`- `Text, Protein` [Proteinaligner pretrain data](https://drive.google.com/file/d/1Ff28ajdyUDQl9JtSNQcUz-6Nbz7FXdEA/view)
- `Pre-training`- `Text, Protein` [ProTAD](https://drive.google.com/file/d/1oyl9JVfEvDk72HdtFaPrRKgUDmI_AFHb/view)
- `Pre-training`- `Text, Protein` [Molinst-SwissProtCLAP](https://huggingface.co/datasets/nwliu/Molinst-SwissProtCLAP)
- `Fine-tuning`- `Text` [BLURB](https://microsoft.github.io/BLURB)
- `Fine-tuning`- `Text` [PubMedQA](https://github.com/pubmedqa/pubmedqa)
- `Fine-tuning`- `Text` [SciQ](https://huggingface.co/datasets/sciq)
- `Fine-tuning`- `Text` [BioASQ](http://participants-area.bioasq.org/datasets)
- `Fine-tuning`- `Text` [MedC-I](https://huggingface.co/datasets/axiong/pmc_llama_instructions)
- `Fine-tuning`- `Molecule` [MoleculeNet](https://moleculenet.org/datasets-1)
- `Fine-tuning`- `Molecule` [MoleculeACE](https://github.com/molML/MoleculeACE)
- `Fine-tuning`- `Molecule` [TDC](https://tdcommons.ai/)
- `Fine-tuning`- `Molecule` [USPTO](https://yzhang.hpc.nyu.edu/T5Chem)
- `Fine-tuning`- `Molecule` [Graph2graph](https://github.com/wengong-jin/iclr19-graph2graph/tree/master/data)
- `Fine-tuning`- `Molecule` [PubChem Molecule Optimization](https://huggingface.co/datasets/yerevann/PubChemForLM)
- `Fine-tuning`- `Protein` [PEER](https://github.com/DeepGraphLearning/PEER_Benchmark)
- `Fine-tuning`- `Protein` [FLIP](https://benchmark.protein.properties)
- `Fine-tuning`- `Protein` [TAPE](https://github.com/songlab-cal/tape)
- `Fine-tuning`- `Text, Molecule` [PubChemSTM](https://huggingface.co/datasets/chao1224/MoleculeSTM/tree/main)
- `Fine-tuning`- `Text, Molecule` [PseudoMD-1M](https://huggingface.co/datasets/SCIR-HI/PseudoMD-1M)
- `Fine-tuning`- `Text, Molecule` [ChEBI-20](https://github.com/blender-nlp/MolT5)
- `Fine-tuning`- `Text, Molecule` [ChEBI-20-MM](https://github.com/AI-HPC-Research-Team/SLM4Mol)
- `Fine-tuning`- `Text, Molecule` [ChEBL-dia](https://github.com/Ellenzzn/ChatMol/tree/main/data/ChEBI-dia)
- `Fine-tuning`- `Text, Molecule` [PCdes](https://github.com/thunlp/KV-PLM)
- `Fine-tuning`- `Text, Molecule` [MoMu](https://github.com/yangzhao1230/GraphTextRetrieval)
- `Fine-tuning`- `Text, Molecule` [PubChemQA](https://github.com/PharMolix/OpenBioMed)
- `Fine-tuning`- `Text, Molecule` [3D-MolT](https://huggingface.co/datasets/Sihangli/3D-MoIT)
- `Fine-tuning`- `Text, Molecule` [MoleculeQA](https://github.com/IDEA-XL/MoleculeQA)
- `Fine-tuning`- `Text, Molecule` [MolTextQA](https://github.com/siddharthal/MolTextQA)
- `Fine-tuning`- `Text, Molecule` [MolOpt-Instructions](https://github.com/blazerye/DrugAssist)
- `Fine-tuning`- `Text, Molecule` [SMolInstruct](https://github.com/OSU-NLP-Group/LLM4Chem)
- `Fine-tuning`- `Text, Molecule` [PubChem324k](https://huggingface.co/datasets/acharkq/PubChem324kV2)
- `Fine-tuning`- `Text, Molecule` [KnowMol-100K](https://huggingface.co/datasets/yzf1102/KnowMol-100K)
- `Fine-tuning`- `Text, Molecule` [MolQA](https://huggingface.co/datasets/liuganghuggingface/Llamole-MolQA)
- `Fine-tuning`- `Text, Molecule` [MolTC](https://huggingface.co/chang04/ddi)
- `Fine-tuning`- `Text, Molecule` [Mol-LLaMA-Instruct](https://huggingface.co/datasets/DongkiKim/Mol-LLaMA-Instruct)
- `Fine-tuning`- `Text, Molecule` [PEIT-LLM](https://pan.baidu.com/s/1VcFvrVHmjBZpL2L_QWt9TQ?pwd=vvts)
- `Fine-tuning`- `Text, Molecule` [SmileyLlama](https://anonymous.4open.science/r/SmileyLlama-gh-D038)
- `Fine-tuning`- `Text, Molecule` [MuMOInstruct](https://huggingface.co/datasets/NingLab/MuMOInstruct)
- `Fine-tuning`- `Text, Molecule` [HiPubChem](https://huggingface.co/datasets/lfhase/HIGHT)
- `Fine-tuning`- `Text, Molecule` [ExDDI](https://drive.google.com/drive/folders/1o2UgoPvMw6hunoewChw0ErumxXWJ4aTX?usp=sharing)
- `Fine-tuning`- `Text, Molecule` [MMP](https://github.com/Cello2195/TransDLM/tree/main/datasets/mmp)
- `Fine-tuning`- `Text, Molecule` [SLM4CRP_with_RTs](https://huggingface.co/datasets/liupf/SLM4CRP_with_RTs)
- `Fine-tuning`- `Text, Molecule, etc` [SciAssess](https://sci-assess.github.io)
- `Fine-tuning`- `Text, Molecule, etc` [DrugBank](https://github.com/SCIR-HI/ArtificiallyR2R)
- `Fine-tuning`- `Text, Molecule, etc` [DARWIN](https://github.com/MasterAI-EAM/Darwin/tree/main/dataset)
- `Fine-tuning`- `Text, Protein` [SwissProt](https://www.expasy.org/resources/uniprotkb-swiss-prot)
- `Fine-tuning`- `Text, Protein` [UniProtQA](https://github.com/PharMolix/OpenBioMed)
- `Fine-tuning`- `Text, Protein` [InstructProtein](-)
- `Fine-tuning`- `Text, Protein` [Open Protein Instructions](https://github.com/baaihealth/opi)
- `Fine-tuning`- `Text, Protein` [PDB-QA](https://osf.io/23azs/files?view_only=185575515e714f4798499bf06513a730)
- `Fine-tuning`- `Text, Protein` [ProteinQA](https://drive.google.com/file/d/1xdiBP-FPMfwpMGBUPAKd0FyRrqQtxAEK/view)
- `Fine-tuning`- `Text, Protein` [Protein2Text-QA](-)
- `Fine-tuning`- `Text, Protein` [CAMEO](https://huggingface.co/datasets/nwliu/CAMEO)
- `Fine-tuning`- `Text, Protein` [Swiss-Prot Curated Triplets](https://drive.google.com/file/d/1n5Ant3S5QE0Yx-DznRa3lannFanc1WB7/view)
- `Fine-tuning`- `Text, Protein` [ProCyon-Instruct](https://huggingface.co/datasets/mims-harvard/ProCyon-Instruct)
- `Fine-tuning`- `Text, Molecule, Protein` [Mol-Instructions](https://huggingface.co/datasets/zjunlp/Mol-Instructions)
- `Fine-tuning`- `Text, Molecule, Protein` [Biology-Instructions](https://github.com/hhnqqq/BiologyInstructions)
- `Benchmark`- `Text` [SciEval](https://github.com/OpenDFM/SciEval)
- `Benchmark`- `Text` [BioInfo-Bench](https://github.com/cinnnna/bioinfo-bench)
- `Benchmark`- `Text` [BioMedEval](https://github.com/tahmedge/llm-eval-biomed)
- `Benchmark`- `Text, Molecule` [ChemLLMBench](https://github.com/ChemFoundationModels/ChemLLMBench)
- `Benchmark`- `Text, Molecule` [AI4Chem](https://github.com/andresilvapimentel/AI4Chem)
- `Benchmark`- `Text, Molecule` [GPTChem](https://github.com/kjappelbaum/gptchem)
- `Benchmark`- `Text, Molecule, etc` [StructChem](https://github.com/ozyyshr/StructChem)
- `Benchmark`- `Text, Molecule` [S¬≤-Bench](https://huggingface.co/datasets/Duke-de-Artois/TOMG-Bench)
- `Benchmark`- `Text, Molecule` [MotifHallu](https://huggingface.co/datasets/lfhase/HIGHT)
- `Benchmark`- `Text, Molecule` [MolCap-Arena](https://github.com/Genentech/molcap-arena)
- `Benchmark`- `Text, Molecule` [ChemCoTBench](https://huggingface.co/datasets/OpenMol/ChemCoTBench)
- `Benchmark`- `Text, Molecule` [MolLangBench](https://huggingface.co/datasets/ChemFM/MolLangBench)
- `Benchmark`- `Text, Molecule` [MolErr2Fix](https://huggingface.co/datasets/YoungerWu/MolErr2Fix)
- `Benchmark`- `Text, Molecule` [MolPuzzle](https://huggingface.co/datasets/kguo2/MolPuzzle_data)
- `Benchmark`- `Text, Protein` [ProteinLMBench](https://huggingface.co/datasets/tsynbio/ProteinLMBench)
- `Benchmark`- `Text, Protein` [Prot-Inst-OOD](https://huggingface.co/datasets/TimeRune/Prot-Inst-OOD)

## Related Resources
### Related Surveys & Evaluations
* Multimodal Pre-training Models of Molecular Representation for Drug Discovery [NSF_2511](https://academic.oup.com/nsr/advance-article/doi/10.1093/nsr/nwaf495/8320534)
* A Comprehensive Survey of Multimodal LLMs for Scientific Discovery [VLM4RWD@NeurIPS 2511](https://openreview.net/forum?id=HSz1Kr5BeC)
* A Survey of Scientific Large Language Models: From Data Foundations to Agent Frontiers [Arxiv_2508](https://arxiv.org/abs/2508.21148)
* A Survey of Large Language Models for Text-Guided Molecular Discovery: from Molecule Generation to Optimization [Arxiv_2505](https://arxiv.org/abs/2505.16094)
* From Generalist to Specialist: A Survey of Large Language Models for Chemistry [Arxiv_2412](https://arxiv.org/abs/2412.19994)
* A Review of Large Language Models and Autonomous Agents in Chemistry [Arxiv_2407](https://arxiv.org/abs/2407.01603)
* A Comprehensive Survey of Scientific Large Language Models and Their Applications in Scientific Discovery [Arxiv 2406](https://arxiv.org/abs/2406.10833)
* Bridging Text and Molecule: A Survey on Multimodal Frameworks for Molecule [Arxiv 2403](https://arxiv.org/abs/2403.13830)
* Bioinformatics and Biomedical Informatics with ChatGPT: Year One Review [Arxiv 2403](https://arxiv.org/abs/2403.15274)
* From Words to Molecules: A Survey of Large Language Models in Chemistry [Arxiv 2402](https://arxiv.org/abs/2402.01439)
* Scientific Language Modeling: A Quantitative Review of Large Language Models in Molecular Science [Arxiv 2402](https://arxiv.org/abs/2402.04119)
* Progress and Opportunities of Foundation Models in Bioinformatics [Arxiv 2402](https://arxiv.org/abs/2402.04286)
* Scientific Large Language Models: A Survey on Biological & Chemical Domains [ACM Computing Surveys](https://arxiv.org/abs/2401.14656)
* The Impact of Large Language Models on Scientific Discovery: a Preliminary Study using GPT-4 [Arxiv 2311](https://arxiv.org/abs/2311.07361)
* Transformers and Large Language Models for Chemistry and Drug Discovery [Arxiv 2310](https://arxiv.org/abs/2310.06083)
* Language models in molecular discovery [Arxiv 2309](https://arxiv.org/abs/2309.16235)
* What can Large Language Models do in chemistry? A comprehensive benchmark on eight tasks [NeurIPS 2309](https://openreview.net/pdf?id=1ngbR3SZHW)
* Do Large Language Models Understand Chemistry? A Conversation with ChatGPT [JCIM 2303](https://pubs.acs.org/doi/10.1021/acs.jcim.3c00285)
* A Systematic Survey of Chemical Pre-trained Models [IJCAI 2023](https://www.ijcai.org/proceedings/2023/0760.pdf)

### Related Workshop
* [Language + Molecules @ ACL 2024 Workshop](https://language-plus-molecules.github.io/)

### Related Repositories
* [MultiPM4Drug](https://github.com/AISciLab/MultiPM4Drug)
* [Awesome-Scientific-Datasets-and-LLMs](https://github.com/open-sciencelab/Awesome-Scientific-Datasets-and-LLMs)
* [LLM4Chemistry](https://github.com/OpenDFM/LLM4Chemistry)
* [LLMs-in-science](https://github.com/ur-whitelab/LLMs-in-science)
* [Awesome-LLM-Centric-Molecular-Discovery](https://github.com/REAL-Lab-NU/Awesome-LLM-Centric-Molecular-Discovery)
* [LLM4ScientificDiscovery](https://github.com/microsoft/LLM4ScientificDiscovery)
* [SLM4Mol](https://github.com/AI-HPC-Research-Team/SLM4Mol)
* [Scientific-LLM-Survey](https://github.com/HICAI-ZJU/Scientific-LLM-Survey)
* [Awesome-Bio-Foundation-Models](https://github.com/apeterswu/Awesome-Bio-Foundation-Models)
* [Awesome-Molecule-Text](https://github.com/Namkyeong/awesome-molecule-text)
* [LLM4Mol](https://github.com/HHW-zhou/LLM4Mol)
* [Awesome-Chemical-Pre-trained-Models](https://github.com/junxia97/awesome-pretrain-on-molecules)
* [Awesome-Chemistry-Datasets](https://github.com/kjappelbaum/awesome-chemistry-datasets)
* [Awesome-Docking](https://github.com/KyGao/awesome-docking)

## Acknowledgements
This repository is contributed and updated by [QizhiPei](https://qizhipei.github.io) and [Lijun Wu](https://apeterswu.github.io). If you have questions, don't hesitate to open an issue or ask me via <qizhipei@ruc.edu.cn> or Lijun Wu via <lijun_wu@outlook.com>. We are happy to hear from you!

## Citations
```
@article{pei2024leveraging,
  title={Leveraging Biomolecule and Natural Language through Multi-Modal Learning: A Survey},
  author={Pei, Qizhi and Wu, Lijun and Gao, Kaiyuan and Zhu, Jinhua and Wang, Yue and Wang, Zun and Qin, Tao and Yan, Rui},
  journal={arXiv preprint arXiv:2403.01528},
  year={2024}
}
```

![Star History Chart](https://api.star-history.com/svg?repos=QizhiPei/Awesome-Biomolecule-Language-Cross-Modeling&type=Date)